<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
	<head>
		<meta charset="utf-8" />
		<title>BCSC2019_S02_C18_Appendix_p477_492_9P</title>
		<link href="BCSC2019_S02_C18_Appendix_p477_492_9P-web-resources/css/idGeneratedStyles.css" rel="stylesheet" type="text/css" />
	</head>
	<body id="BCSC2019_S02_C18_Appendix_p477_492_9P" lang="en-US">
		<div id="_idContainer000" class="Basic-Text-Frame">
			<p class="chapter-number"><span class="cn-chap">Appendix</span></p>
			<p class="chapter-title ParaOverride-1">Genetics Glossary</p>
			<p class="glossary-entry ParaOverride-2"><span class="glossary-term">AAV (adeno-&#173;associated virus) vector gene therapy</span> A form of gene therapy in which the target gene is inserted into the cell by an AAV vector.</p>
			<p class="glossary-entry"><span class="glossary-term">Allele</span> Alternative form of a gene that may occupy a given locus on a pair of chromosomes. Clinical traits, gene products, and disorders are said to be <span class="glossary-entry_italic">allelic</span> if their genes are determined to be at the same locus and <span class="glossary-entry_italic">nonallelic</span> if they are determined to reside at dif&#173;fer&#173;ent loci.</p>
			<p class="glossary-entry"><span class="glossary-term">Allelic heterogeneity</span> Refers to the capability of dif&#173;fer&#173;ent alleles at the same locus to produce an abnormal phenotype.</p>
			<p class="glossary-entry"><span class="glossary-term">Aneuploidy</span> An abnormal number of chromosomes.</p>
			<p class="glossary-entry"><span class="glossary-term">Anticipation</span> The occurrence of a dominantly inherited disease at an &#173;earlier age (often with greater severity) in subsequent generations. Now known to occur with expansion of trinucleotide repeats. Observed, for example, in fragile X syndrome, myotonic dystrophy, and Huntington disease.</p>
			<p class="glossary-entry"><span class="glossary-term">Antisense strand of DNA</span> The strand of double-&#173;stranded DNA that serves as a template for RNA transcription. Also called the <span class="glossary-entry_italic">noncoding,</span> or <span class="glossary-entry_italic">transcribed,</span> strand.</p>
			<p class="glossary-entry"><span class="glossary-term">Apoptosis</span> The pro&#173;cess by which internal or external messages trigger expression of specific genes and their products, resulting in the initiation of a series of cellular events that involve fragmentation of the cell nucleus, dissolution of cellular structure, and orderly cell death. Unlike traumatic cell death, apoptosis results in the death of individual cells rather than clusters of cells and does not lead to the release of inflammatory intracellular products. Also called <span class="glossary-entry_italic">programmed cell death (PCD).</span></p>
			<p class="glossary-entry"><span class="glossary-term">Assortative mating</span> Mating between individuals with a preference for or against a specific phenotype or genotype; that is, nonrandom mating.</p>
			<p class="glossary-entry"><span class="glossary-term">Autosome</span> Any chromosome other than the sex (X and Y) chromosomes. The normal &#173;human has 22 pairs of autosomes.</p>
			<p class="glossary-entry"><span class="glossary-term">Barr body</span> Inactive X chromosome found in the nucleus of some female somatic cells.</p>
			<p class="glossary-entry"><span class="glossary-term">Base pair (bp)</span> Two complementary nitrogen bases that are paired in double-&#173;stranded DNA. Used as a unit of physical distance or length of a sequence of nucleotides.</p>
			<p class="glossary-entry"><span class="glossary-term">Biobank</span> A repository that stores biological samples for use in research. The UK Biobank is a collection of DNA samples from more than 500,000 &#173;people and, for many participants, ophthalmic phenotype data.</p>
			<p class="glossary-entry"><span class="glossary-term">Carrier</span> An individual who has a pair of genes consisting of 1 normal and 1 abnormal, or <span class="glossary-entry_italic">mutant,</span> gene. Usually, such individuals are by definition phenotypically “normal,” although in certain disorders biochemical evidence of a deficient or defective gene product may be pre&#173;sent. Occasionally, carriers of an X-&#173;linked disorder may show partial expression of a ge&#173;ne&#173;tic trait.</p>
			<p class="glossary-entry"><span class="glossary-term">Cell-&#173;free fetal DNA (cffDNA)</span> Fetal DNA that circulates freely in the maternal blood and can be sampled by phlebotomy, providing a noninvasive method of prenatal diagnosis. Sometimes called <span class="glossary-entry_italic">noninvasive prenatal screening.</span></p>
			<p class="glossary-entry"><span class="glossary-term">Centimorgan (cM)</span> A mea&#173;sure of the crossover frequency between linked genes; 1 cM equals 1% recombination and represents a physical distance of approximately 1 million bp.</p>
			<p class="glossary-entry"><span class="glossary-term">Centromere</span> The constricted region of a chromosome where &#173;sister chromatids are joined. It is also the site of attachment to spindle fibers during mitosis and meiosis and is impor&#173;tant in the movement of chromosomes to the poles of the dividing cell.</p>
			<p class="glossary-entry"><span class="glossary-term">Chorionic villus sampling (CVS)</span> Transcervical procedure in which chorionic villi are retrieved with a flexible suction catheter and used in studies to establish a prenatal diagnosis.</p>
			<p class="glossary-entry"><span class="glossary-term">Chromatid</span> One of the duplicate arms (also called <span class="glossary-entry_italic">&#173;sister chromatids</span>) of chromosomes that are created &#173;after DNA replication during mitosis or the first division of meiosis.</p>
			<p class="glossary-entry"><span class="glossary-term">Chromatin</span> The complex of DNA and proteins that is pre&#173;sent in chromosomes. Chromatin is found in 2 va&#173;ri&#173;e&#173;ties: euchromatin and heterochromatin. Euchromatin is a lightly packed form of chromatin that is often &#173;under active transcription. Heterochromatin consists primarily of genet&#173;ically inactive satellite sequences such as centromeres, telomeres, and the Barr body.</p>
			<p class="glossary-entry"><span class="glossary-term">Clinical heterogeneity</span> The production of dif&#173;fer&#173;ent phenotypes by dif&#173;fer&#173;ent mutations at the same locus. Examples include macular dystrophy and retinitis pigmentosa from differing mutations of peripherin/<span class="glossary-entry_italic">RDS</span> and Crouzon, Pfeiffer, and Apert syndromes from differing mutations of <span class="glossary-entry_italic">FGFR2. </span>See<span class="glossary-entry_italic"> Genetic heterogeneity.</span></p>
			<p class="glossary-entry"><span class="glossary-term">Clinical Laboratory Improvement Amendments (CLIA)</span> A federal program that sets standards for clinical laboratory testing in the United States. Clinicians are advised to use a CLIA-&#173;certified laboratory to provide results to their patients.</p>
			<p class="glossary-entry"><span class="glossary-term">Codominance</span> Simultaneous expression of both alleles of a heterozygous locus (eg, ABO blood groups).</p>
			<p class="glossary-entry"><span class="glossary-term">Codon</span> A sequence of 3 adjacent nucleotides that forms the basic unit of the ge&#173;ne&#173;tic code. The DNA molecule is a chain of nucleotide bases read in units of 3 bases <span class="glossary-entry_italic">(triplets),</span> each of which &#173;will translate through messenger RNA (mRNA) into an amino acid. Thus, each triplet codon specifies a single amino acid.</p>
			<p class="glossary-entry"><span class="glossary-term">Complementary DNA (cDNA)</span> DNA created by the action of reverse transcriptase from mRNA. In contrast to genomic DNA, cDNA does not have introns.</p>
			<p class="glossary-entry"><span class="glossary-term">Complex ge&#173;ne&#173;tic disorder (multigene disorder)</span> A trait or medical disorder (eg, age-&#173;related macular degeneration) that does not follow mendelian patterns of inheritance. Close relatives have a higher risk of the disorder, suggesting that the state is determined by genes at multiple loci (and that environmental &#173;factors may be involved).</p>
			<p class="glossary-entry"><span class="glossary-term">Compound heterozygote</span> An individual with 2 dif&#173;fer&#173;ent, abnormal alleles at the same locus, 1 on each homologous chromosome.</p>
			<p class="glossary-entry"><span class="glossary-term">Congenital</span> Pre&#173;sent at birth. The term has no implications about the origin of the congenital feature, which could be ge&#173;ne&#173;tic or environmental.</p>
			<p class="glossary-entry"><span class="glossary-term">Consanguinity</span> The ge&#173;ne&#173;tic relationship between individuals who are the descendants of a sexual &#173;union between blood relatives; in other words, individuals who share a recent common ancestor (eg, offspring of a marriage between cousins).</p>
			<p class="glossary-entry"><span class="glossary-term">Conservation</span> Refers to the presence of a similar ge&#173;ne&#173;tic sequence or nucleotide position among dif&#173;fer&#173;ent species at 1 gene or related genes of similar sequence. In such cases, the sequence or position is said to be <span class="glossary-entry_italic">conserved</span> or to show conservation.</p>
			<p class="glossary-entry"><span class="glossary-term">Copy number variation</span> A structural variation in the genome resulting from deleted or inserted nucleotides, exons, or genes (also referred to as <span class="glossary-entry_italic">indels</span>). Indels have been used as DNA markers to locate genes but may also cause disease by adding amino acids (eg, in triplet repeat disorders) or may affect gene expression (eg, the expression of color opsins on the X chromosome).</p>
			<p class="glossary-entry"><span class="glossary-term">CRISPR–&#173;Cas9 (clustered, regularly interspaced, short palindromic repeats–&#173;CRISPR-&#173;associated protein 9)</span> A new form of genome editing used to correct point mutations in the DNA sequence of cells.</p>
			<p class="glossary-entry"><span class="glossary-term">Crossing over</span> A pro&#173;cess in which matching segments of homologous chromosomes (chromatids) break and are exchanged to the other chromosome, where they are reconnected to the other chromosome by repair of the breaks. Crossing over is a regular event in meiosis but occurs only rarely in mitosis. See<span class="glossary-entry_italic"> Recombination.</span></p>
			<p class="glossary-entry"><span class="glossary-term">Database of Genotypes and Phenotypes (dbGaP)</span> Funded by the US National Institutes of Health, a cata&#173;log of ge&#173;ne&#173;tic information (genotypes) linked to clinical information (phenotypes) from results of genome-&#173;wide association studies (GWAS).</p>
			<p class="glossary-entry"><span class="glossary-term">Degeneracy of ge&#173;ne&#173;tic code</span> The redundancy of the ge&#173;ne&#173;tic code stemming from the fact that most of the 20 amino acids are coded for by more than 1 of the 64 pos&#173;si&#173;ble triplet codons. The ge&#173;ne&#173;tic code is termed <span class="glossary-entry_italic">degenerate.</span></p>
			<p class="glossary-entry"><span class="glossary-term">Digenic inheritance</span> Simultaneous inheritance of 2 nonallelic mutant genes, giving rise to a ge&#173;ne&#173;tic disorder in which inheritance of only 1 of the 2 is insufficient to cause disease. Digenic inheritance is the simplest form of polygenic inheritance. An example is retinitis pigmentosa caused by simultaneous inheritance in the heterozygous state of other&#173;wise tolerable mutations of both the <span class="glossary-entry_italic">ROM1</span> and peripherin/<span class="glossary-entry_italic">RDS</span> genes.</p>
			<p class="glossary-entry"><span class="glossary-term">Diploid</span> Refers to the number of chromosomes in most somatic cells, which in &#173;humans is 46. The diploid number is twice the haploid number (the number of chromosomes in gametes).</p>
			<p class="glossary-entry"><span class="glossary-term">Direct-&#173;to-&#173;consumer ge&#173;ne&#173;tic tests</span> Ge&#173;ne&#173;tic tests marketed directly to consumers. An individual &#173;orders a test kit directly from a ge&#173;ne&#173;tic testing laboratory and mails a tissue sample (saliva or blood) back to the laboratory, which runs a series of DNA tests, usually single nucleotide polymorphisms (SNPs) or, in some cases, sequencing. &#173;These DNA tests may be specific to certain genes or diseases or may involve a large panel of genes and diseases. Lack of counseling, quality control, and appropriate scientific interpretation of data are major challenges to direct-&#173;to-&#173;consumer testing.</p>
			<p class="glossary-entry"><span class="glossary-term">Dizygotic (DZ) twins</span> Occur when 2 individual ova (eggs) are fertilized by separate sperm, resulting in nonidentical (fraternal) twins, who share 50% of their DNA sequence.</p>
			<p class="glossary-entry"><span class="glossary-term">DNA code</span> The sequences of DNA trinucleotides corresponding to the amino acids.</p>
			<p class="glossary-entry"><span class="glossary-term">Dominant</span> Refers to an allele that is expressed in the phenotype when inherited along with a normal allele. See <span class="glossary-entry_italic">Recessive.</span></p>
			<p class="glossary-entry"><span class="glossary-term">Dominant medical disorder</span> A distinctive disease state that occurs in an individual with a genotype that is heterozygous for a dominant disease-&#173;producing allele. Homozygotes for dominant disease-&#173;producing alleles are rare and are usually more severely affected than heterozygotes. By definition, normal dominant traits produce the same phenotype in both the heterozygous and the homozygous states.</p>
			<p class="glossary-entry"><span class="glossary-term">Dominant negative mutation</span> An autosomal dominant mutation that disrupts the function of the normal or wild-&#173;type allele in the heterozygous state, producing a phenotype approaching that of the homozygous mutant.</p>
			<p class="glossary-entry"><span class="glossary-term">ENCODE (ENCyclopedia Of DNA Ele&#173;ments)</span> A cata&#173;log of functional ele&#173;ments in the &#173;human genome (see <a href="https://www.genome.gov/encode/">https://www.genome.gov/encode/</a>).</p>
			<p class="glossary-entry"><span class="glossary-term">Endonuclease</span> A phosphodiester-&#173;cleaving enzyme, usually derived from bacteria, that cuts nucleic acids at internal positions. Restriction endonucleases cut at specific recognition sites determined by the occurrence of a specific recognition sequence of 4, 5, or 6 bp. Endonuclease specificity may also be confined to substrate conformation, nucleic acid species (DNA, RNA), and the presence of modified nucleotides.</p>
			<p class="glossary-entry"><span class="glossary-term">Enhancer</span> Any sequence of DNA upstream or downstream of the coding region that acts in <span class="glossary-entry_italic">cis</span> (ie, on the same chromosome) to increase (or, as a negative enhancer, to decrease) the rate of transcription of a nearby gene. Enhancers may display tissue specificity and act over considerable distances.</p>
			<p class="glossary-entry"><span class="glossary-term">Epige&#173;ne&#173;tics/epigenomics</span> The study of modifications of the expression of the ge&#173;ne&#173;tic code by &#173;factors that may themselves be genet&#173;ically or environmentally influenced. In &#173;these cases, gene expression is affected without alteration of the DNA sequence. Examples of such &#173;factors include cytosine methylation and histone formation.</p>
			<p class="glossary-entry"><span class="glossary-term">Eukaryote</span> Organisms whose DNA is located within a nucleus (includes all multicellular and higher unicellular organisms).</p>
			<p class="glossary-entry"><span class="glossary-term">Exome sequencing</span> A strategy to selectively sequence the coding regions of the genome as a less expensive alternative to whole-&#173;genome sequencing. Approximately 180,000 exons constitute 1% of the &#173;human genome, or approximately 30 megabases (Mb).</p>
			<p class="glossary-entry"><span class="glossary-term">Exon</span> A coding sequence of DNA is represented in the mature mRNA product. See <span class="glossary-entry_italic">Intron.</span></p>
			<p class="glossary-entry"><span class="glossary-term">Expressed-&#173;sequence tag (EST)</span> A partial sequence of a gene that uniquely identifies the gene’s message. &#173;These tags are useful in reverse transcriptase polymerase chain reaction (RT-&#173;PCR) detection for determining the expression levels of large numbers of genes in parallel reactions.</p>
			<p class="glossary-entry"><span class="glossary-term">Expressivity</span> The variation in clinical manifestation among individuals with a par&#173;tic&#173;u&#173;lar genotype, usually a dominant medical disorder. The variability may be a difference in &#173;either age at onset (manifestation) or severity. See <span class="glossary-entry_italic">Penetrance.</span></p>
			<p class="glossary-entry"><span class="glossary-term">Fragile sites</span> Reproducible sites of secondary constrictions, gaps, or breaks in chromatids. Fragile sites are transmitted as mendelian codominant traits and are not usually associated with abnormal phenotype. The most notable exceptions are the association of fragile X chromosomes with X-&#173;linked cognitive impairment and with postpubertal macro-&#173;orchidism (fragile X syndrome). See <span class="glossary-entry_italic">Trinucleotide repeat expansion/contraction.</span></p>
			<p class="glossary-entry"><span class="glossary-term">Frameshift mutation (framing error, frameshift)</span> A deletion or insertion of a nucleotide or a number of nucleotides not divisible by 3 that results in a loss of the normal sequences of triplets, causing the new sequence to code for entirely dif&#173;fer&#173;ent amino acids from the original. The mutation usually leads to the eventual chance formation of a <span class="glossary-entry_italic">stop codon.</span></p>
			<p class="glossary-entry"><span class="glossary-term">Gene</span> The segment of DNA and its associated regulatory ele&#173;ments coding for a single trait, usually a single polypeptide or mRNA. The definition was expanded to include any expressed sequence of nucleotides that has functional significance, including DNA sequences that govern the transcription of a gene (promoter sequences immediately upstream of the gene or enhancer sequences that may be more distantly located).</p>
			<p class="glossary-entry"><span class="glossary-term">Ge&#173;ne&#173;tic heterogeneity</span> The production of a phenotype (or apparently similar phenotypes) by dif&#173;fer&#173;ent ge&#173;ne&#173;tic entities. Refers to ge&#173;ne&#173;tic disorders that are found to be two or more fundamentally distinct entities. See <span class="glossary-entry_italic">Clinical heterogeneity.</span></p>
			<p class="glossary-entry"><span class="glossary-term">Ge&#173;ne&#173;tic Information Nondiscrimination Act (GINA)</span> A law enacted by the US Congress in 2008 to prohibit the improper use of ge&#173;ne&#173;tic information by health insurance providers and employers.</p>
			<p class="glossary-entry"><span class="glossary-term">Genome</span> The sum of the ge&#173;ne&#173;tic material of a cell or an organism.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Genome-&#173;wide association studies (GWAS)</span> Research studies that examine the associations between single nucleotide polymorphisms (SNPs) and traits or diseases by comparing the DNA of a group of &#173;people with a par&#173;tic&#173;u&#173;lar disease (cases) and another, similar group without that disease (controls). Hundreds of thousands of SNPs are read on arrays in studies designed to find common ancestral mutations that contribute risk for disease.</p>
			<p class="glossary-entry"><span class="glossary-term">Genomic imprinting</span> Imprinting is an epige&#173;ne&#173;tic phenomenon that &#173;causes genes to be expressed in a parent-&#173;of-&#173;origin-&#173;specific manner. It involves DNA methylation and histone methylation without altering the ge&#173;ne&#173;tic sequence. &#173;These epige&#173;ne&#173;tic marks are established <span class="glossary-entry_italic">(imprinted)</span> in the germline (sperm or egg cells) of the parents and are maintained through mitotic cell divisions in the somatic cells of an offspring. Appropriate imprinting of certain genes is impor&#173;tant for normal development. Diseases resulting from abnormalities of genomic imprinting include Angelman syndrome and Prader-&#173;Willi syndrome.</p>
			<p class="glossary-entry"><span class="glossary-term">Genomics</span> The study of the genome. Names for the fields of <span class="glossary-entry_italic">transcriptomics, proteomics,</span> and <span class="glossary-entry_italic">metabolomics</span> &#173;were coined in a similar fashion.</p>
			<p class="glossary-entry"><span class="glossary-term">Genotype</span> The ge&#173;ne&#173;tic constitution of an organism. Also, the specific set of 2 alleles inherited at a locus.</p>
			<p class="glossary-entry"><span class="glossary-term">Germinal mosaicism</span> The occurrence of 2 populations of gametes in an individual, 1 population with a normal allele and the other with a disease-&#173;producing mutant gene. Of new cases of some autosomal dominant diseases (eg, osteogenesis imperfecta), 5%–10% are thought to result from germinal mosaicism; offspring of the affected parent are at significant risk for the same disease.</p>
			<p class="glossary-entry"><span class="glossary-term">Haploid</span> Half the number of chromosomes in most somatic cells; equal to the number of chromosomes in gametes. In &#173;humans, the haploid number is 23. Also used to denote the state in which only 1 of a pair or set of chromosomes is pre&#173;sent. See <span class="glossary-entry_italic">Diploid.</span></p>
			<p class="glossary-entry"><span class="glossary-term">Haploid insufficiency (haploinsufficiency)</span> The condition of dominant ge&#173;ne&#173;tic disease caused by a reduction in the gene product to levels that are insufficient to produce the desired function of the protein. For example, aniridia and Waardenburg syndrome result from insufficiency of the single functional copy of the <span class="glossary-entry_italic">PAX6</span> and <span class="glossary-entry_italic">PAX3</span> genes, respectively, to activate transcription of the genes that they normally control.</p>
			<p class="glossary-entry"><span class="glossary-term">Haplotype</span> A series of contiguous alleles along the length of a single chromosome that may be inherited as a block. Also referred to as a <span class="glossary-entry_italic">haploblock.</span></p>
			<p class="glossary-entry"><span class="glossary-term">HapMap</span> The haplotype map of the &#173;human genome describing the common patterns of &#173;human ge&#173;ne&#173;tic variation. A key resource in finding variants affecting health and disease.</p>
			<p class="glossary-entry"><span class="glossary-term">Hemizygous (hemizygote)</span> Having only 1 allele at a locus; usually refers to X-&#173;linked loci in males, who normally have only 1 set of X-&#173;linked genes. An individual who is missing an entire chromosome or a segment of 1 chromosome is considered hemizygous for the genes on the homologous chromosome. See <span class="glossary-entry_italic">Heterozygous, Homozygous.</span></p>
			<p class="glossary-entry"><span class="glossary-term">Hereditary</span> Genet&#173;ically transmitted or capable of being genet&#173;ically transmitted from parent to offspring. Not quite synonymous with <span class="glossary-entry_italic">heritable,</span> which implies the ability to be transmitted to the next generation but does not intrinsically connote inheritance from the prior generation.</p>
			<p class="glossary-entry ParaOverride-4"><span class="glossary-term">Heteroplasmy</span> The presence of two or more dif&#173;fer&#173;ent populations of mitochondria within a cell, each population carry&#173;ing a dif&#173;fer&#173;ent allele (or the presence or absence of a mutation) at a given locus.</p>
			<p class="glossary-entry ParaOverride-4"><span class="glossary-term">Heterozygous (heterozygote)</span> Having 2 unlike alleles at a par&#173;tic&#173;u&#173;lar locus. See <span class="glossary-entry_italic">Hemizygous, Homozygous.</span></p>
			<p class="glossary-entry ParaOverride-4"><span class="glossary-term">Homeobox</span> A conserved 180-bp sequence of DNA, first detected within homeobox genes (also known as <span class="glossary-entry_italic">homeotic selector genes</span>), that helps determine the cell’s fate.</p>
			<p class="glossary-entry ParaOverride-4"><span class="glossary-term">Homeobox genes</span> Transcription &#173;factor genes that regulate the activity or expression of other genes, eventually guiding the embryonic development of cells into body segments, body parts, and specialized organ systems. Examples are the <span class="glossary-entry_italic">HOX</span> and <span class="glossary-entry_italic">PAX</span> families of developmental genes. Whereas <span class="glossary-entry_italic">HOX</span> genes are involved in early body plan organ&#173;ization, <span class="glossary-entry_italic">PAX</span> genes are involved in somewhat &#173;later organ and body part development. See the discussion of homeobox genes in <span class="glossary-entry_xref-local">Chapter&#160;4</span>.</p>
			<p class="glossary-entry ParaOverride-4"><span class="glossary-term">Homologous chromosomes</span> The 2 members of a matched pair of (&#173;sister) chromosomes, <br />1 derived from each parent, that have the same gene loci, but not necessarily the same alleles, in the same order.</p>
			<p class="glossary-entry ParaOverride-4"><span class="glossary-term">Homoplasmy</span> The presence of a single population of mitochondria within a cell, each carry&#173;ing the same allele (or the same presence or absence of a mutation) at a given locus.</p>
			<p class="glossary-entry ParaOverride-4"><span class="glossary-term">Homozygous (homozygote)</span> Having 2 identical alleles at a par&#173;tic&#173;u&#173;lar locus in the diploid genome. The term is sometimes misused to refer to <span class="glossary-entry_italic">compound heterozygote</span> (see &#173;earlier entry).</p>
			<p class="glossary-entry ParaOverride-4"><span class="glossary-term">&#173;Human Genome Proj&#173;ect (HGP)</span> The international scientific research proj&#173;ect that identified and mapped the approximately 20,000–25,000 &#173;human genes. A working draft of the genome was announced in 2000, and the genome was completed in 2003.</p>
			<p class="glossary-entry ParaOverride-4"><span class="glossary-term">Hybridization</span> The bonding (by Watson-&#173;Crick base pairing) of single-&#173;stranded DNA or RNA into double-&#173;stranded DNA or RNA. The ability of stretches of DNA or RNA to hybridize with one another is highly dependent on complementarity of the base-&#173;pair sequence.</p>
			<p class="glossary-entry ParaOverride-4"><span class="glossary-term">Induced pluripotent stem cells (iPSCs)</span> Adult cells such as skin fibroblasts can be induced to produce pluripotent stem cells by using transcription &#173;factors. iPSCs hold &#173;great promise in the field of regenerative medicine and can be used to create &#173;every type of cell (eg, ret&#173;i&#173;nal cells that could be used to replace cells lost to damage or disease). &#173;Because iPSCs are derived from adult tissues, they not only bypass the need for embryos but also can be made in a patient-&#173;matched manner, which means that each individual could have his or her own pluripotent stem cell line. &#173;These autologous cells reduce the risk of immune rejection. iPSCs are also used for research purposes; for example, ret&#173;i&#173;nal cells that are difficult to obtain from patients can be created for research purposes (“disease in a dish”).</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Intron</span> A noncoding segment of DNA that is transcribed into heterogeneous RNA but is ultimately removed from the transcript by splicing together the sequences on &#173;either side of it (exons) when mature mRNA is produced. See <span class="glossary-entry_italic">Exon.</span></p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Karyotype</span> An image of an individual’s chromosome set obtained from a single somatic cell and arranged in a standard pattern in pairs by size, shape, band pattern, and other identifiable physical features.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Kilobase (kb)</span> 1000 bp of DNA or 1000 bases of single-&#173;stranded RNA.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Library</span> A complete set of clones presumably including all ge&#173;ne&#173;tic material of interest from an organism, tissue, or specific cell type at a specified stage of development. A <span class="glossary-entry_italic">genomic library</span> contains cloned DNA fragments from the entire genome; a <span class="glossary-entry_italic">cDNA library</span> contains fragments of cloned DNA generated by reverse transcription from mRNA. Genomic libraries are useful sources to search for genes, whereas cDNA libraries provide information about expression within the source cell or tissue.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Linkage</span> A concept that refers to loci rather than to the alleles that reside on &#173;those loci. Exists when the loci of 2 genes or DNA sequences are physically close enough to each other on the same chromosome that alleles at the 2 loci do not assort in&#173;de&#173;pen&#173;dently at meiosis but tend to be inherited together.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Linkage disequilibrium</span> Alleles residing at loci that are close together in the genome are inherited together through many generations &#173;because the close physical distance makes crossover between the loci extremely unlikely. Alleles in linkage disequilibrium are pre&#173;s&#173;ent in subpopulations (eg, &#173;those with a given disease) at a frequency higher or lower than expected based on chance alone.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Locus</span> The physical site on a chromosome occupied by a par&#173;tic&#173;u&#173;lar gene. The term is often colloquially used interchangeably with <span class="glossary-entry_italic">gene.</span></p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Locus heterogeneity</span> The production of a similar disease or trait (phenotype) by mutations in genes at dif&#173;fer&#173;ent chromosomal loci, for example, X-linked retinitis pigmentosa resulting from <span class="glossary-entry_italic">RP2</span> at 1 locus, Xp11, and <span class="glossary-entry_italic">RP3</span> at another locus, Xp21. However, only 1 mutant <span class="glossary-entry_italic">locus</span> is needed for the phenotype to manifest.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">LOD (</span><span class="glossary-term_italic">l</span><span class="glossary-term">ogarithm of</span> <span class="glossary-term_italic">od</span><span class="glossary-term">ds, or logarithm of the likelihood ratio)</span> A statistical method that tests &#173;whether a set of linkage data indicates that 2 loci are linked or unlinked. The <span class="glossary-entry_italic">LOD score</span> is the logarithm to the base 10 of the odds favoring linkage. By convention, an LOD score greater than 3 (1000:1 odds in &#173;favor of linkage) is generally accepted as proof of linkage.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Lyonization</span> Inactivation of genes on &#173;either the maternally or the paternally derived X chromosome in somatic cells. The timing of inactivation is variable but may occur around the time of implantation. First proposed by ge&#173;ne&#173;ticist Mary Lyon.</p>
			<p class="glossary-entry"><span class="glossary-term">Manhattan plot</span> A type of plot used in genome-&#173;wide association studies (GWAS). Genomic coordinates are displayed along the x-&#173;axis, with the negative logarithm of the association <span class="glossary-entry_italic">P</span> value for each single nucleotide polymorphism (SNP) displayed on the y-&#173;axis. The strongest associations appear as peaks, calling to mind the skyline of New York City.</p>
			<p class="glossary-entry ParaOverride-5"><span class="glossary-term">Meiosis</span> The special form of cell division that occurs in germ cells by which gametes of haploid chromosomal number are created. Each of the chromatids, which are clearly vis&#173;i&#173;ble by prophase, contains a long double helix of DNA associated with histones and other chromosomal proteins. At anaphase, the chromatids separate at the centromere and migrate to each half of the dividing cell; thus, each &#173;daughter cell receives an identical set of chromatids (which become the chromosomes for that cell). During the first, or <span class="glossary-entry_italic">reduction,</span> division of meiosis, the chromatids of homologous chromosomes undergo crossover (during the diplotene phase), and the number of chromosomes is reduced to the haploid number by the separation of homologous chromosomes (with duplicate chromatids) to each &#173;daughter cell. During the second division of meiosis, the &#173;sister chromatids separate to form the haploid set of chromosomes of each gamete.</p>
			<p class="glossary-entry ParaOverride-5"><span class="glossary-term">Mendelian disorder (single-&#173;gene disorder)</span> A trait or medical disorder that follows patterns of inheritance suggesting the state is determined by a gene at a single locus (eg, autosomal dominant, autosomal recessive, or X-&#173;linked recessive inheritance).</p>
			<p class="glossary-entry ParaOverride-5"><span class="glossary-term">Methylation</span> The attachment of methyl groups to DNA at CpG (cytosine-&#173;phosphate-&#173;guanine) sites. CpG islands are associated with the promoters of many genes, and &#173;there is an inverse relationship between CpG methylation and transcriptional activity.</p>
			<p class="glossary-entry ParaOverride-5"><span class="glossary-term">MicroRNA (miRNA)</span> Small single-&#173;stranded RNA fragment (of approximately 22 nucleotides) that directly interacts with target mRNA through complementary base pairing and inhibits translation of the target genes. miRNA modifies gene expression at transcriptional and posttranscriptional levels.</p>
			<p class="glossary-entry ParaOverride-5"><span class="glossary-term">Microsatellites (eg, dinucleotide or trinucleotide repeats)</span> Tandemly repeated segments scattered throughout the genome of varying numbers of 2–4 nucleotides in a row, for example, a stretch of consecutive CA combinations of bases (NNNCACACACACACACACACACANNN or [CA]<span class="glossary-entry_subscript CharOverride-1">10</span>, where N is any base) in a DNA strand. The repeats are inherently unstable and can undergo mutation at a rate of up to 10%. Defects of some microsatellites are associated with cancer and insulin-&#173;dependent diabetes mellitus, although most have no known biological significance. Other terms used are <span class="glossary-entry_italic">variable number of tandem repeats (VNTR)</span> and <span class="glossary-entry_italic">variable tandem repeats (VTR).</span> The highly variable nature of the number of repeats provides information useful as markers for establishing linkage to disease loci.</p>
			<p class="glossary-entry ParaOverride-5"><span class="glossary-term">Missense mutation</span> A mutation, often the change of a single nucleotide, that results in the substitution of 1 amino acid for another in the final gene product.</p>
			<p class="glossary-entry ParaOverride-5"><span class="glossary-term">Mitosis</span> The ordinary form of cell division, which results in &#173;daughter cells identical in chromosomal number to the parent cell.</p>
			<p class="glossary-entry"><span class="glossary-term">Monozygotic (MZ) twins</span> Identical twins who share 100% of their DNA sequence.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Mosaicism</span> The presence of two or more populations of cells with dif&#173;fer&#173;ent genotypes in 1 individual who has developed from a single fertilized egg. Mosaicism can occur in the cells of 1 part of the body (eg, the ret&#173;ina) or in all cells of an individual.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Multifactorial inheritance</span> The combined operation of several unspecified ge&#173;ne&#173;tic and environmental &#173;factors in the inheritance of a par&#173;tic&#173;u&#173;lar trait or disease. See <span class="glossary-entry_italic">Polygenic inheritance.</span></p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Mutation</span> Any alteration of a gene or ge&#173;ne&#173;tic material from its “natu&#173;ral” state, regardless of &#173;whether the change has a positive, neutral, or negative effect.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Next-&#173;generation sequencing (massively parallel DNA sequencing)</span> Sequencing technology that speeds the pro&#173;cess, producing thousands or millions of DNA sequences at once. This technology has allowed rapid, large-&#173;scale DNA sequencing at much lower cost.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Nondisjunction</span> Failure of 2 chromosomes to separate during meiosis or mitosis.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Nonsense mutation</span> Any mutation that &#173;either results directly in the formation of a stop codon or creates a stop codon in the downstream sequence &#173;after a frameshift mutation. This typically results in a truncated nonfunctional protein.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Nucleoside</span> The combination of a nitrogen-&#173;containing base and a 5-&#173;carbon sugar. The 5 nucleosides are adenosine (A), guanosine (G), cytidine (C), uridine (U), and thymidine (T). Note that the abbreviations are the same as &#173;those for the nitrogen bases that characterize the nucleoside.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Nucleosome</span> The primary unit of chromatin, consisting of a 146-bp sequence of DNA wrapped twice around a core composed of 8 histone molecules.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Nucleotide</span> The combination of a nucleoside and one or more phosphate groups. <span class="glossary-entry_italic">Purine nucleotides</span> have a nitrogen-&#173;containing base of adenine (A) and guanine (G) in DNA or RNA. <span class="glossary-entry_italic">Pyrimidine nucleotides</span> have a nitrogen-&#173;containing base of thymine (T) and cytosine (C) in DNA and uracil (U) in RNA.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">OMIM (Online Mendelian Inheritance in Man)</span> An online database of genes and ge&#173;ne&#173;tic disorders (see <a href="https://omim.org">https://omim.org</a>).</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Oncogene</span> A gene that, when dysregulated, is capable of transforming cells into a neoplastic phenotype characterized by loss of growth control and/or tumorigenesis in a suitable host or site. In many cases, cancer is caused by the growth-&#173;stimulating effects of increased expression, protein activation, or aberrant regulation of transcription &#173;factors required for normal growth. Certain oncogenes are produced by chromosomal translocations of normal transcription &#173;factor genes to regions adjacent to more abundantly expressed genes, causing inappropriate excessive expression. See <span class="glossary-entry_italic">Tumor suppressor genes.</span></p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">1000 Genomes Proj&#173;ect</span> An international collaboration, launched in 2008, to compile the most detailed public cata&#173;log of &#173;human ge&#173;ne&#173;tic variation.</p>
			<p class="glossary-entry"><span class="glossary-term">Open reading frame (ORF)</span> Any part of the genome that could be translated into a protein sequence starting with an initiation (start) codon and ending with a stop codon.</p>
			<p class="glossary-entry"><span class="glossary-term">p arm</span> The short arm of a chromosome in relation to the centromere. From <span class="glossary-entry_italic">petit.</span> See <span class="glossary-entry_italic">q </span><span class="glossary-entry_italic">arm.</span></p>
			<p class="glossary-entry"><span class="glossary-term">Penetrance</span> The probability of detecting the clinical expression of a gene or combination of genes when they are pre&#173;sent. If the penetrance of a par&#173;tic&#173;u&#173;lar condition is less than 100%, not all individuals who carry the responsible gene variant &#173;will develop the condition. Nonpenetrance is the lack of phenotypic evidence of the genotype. See <span class="glossary-entry_italic">Expressivity.</span></p>
			<p class="glossary-entry"><span class="glossary-term">Personalized ge&#173;ne&#173;tics</span> The use of personal genomic data to determine care, including drug treatment, for an individual patient.</p>
			<p class="glossary-entry"><span class="glossary-term">Pharmacoge&#173;ne&#173;tics</span> The study of the influence of ge&#173;ne&#173;tic variation on drug efficacy or toxicity, focusing on single genes. The term is often used interchangeably with <span class="glossary-entry_italic">pharmacogenomics.</span></p>
			<p class="glossary-entry"><span class="glossary-term">Pharmacogenomics</span> The study of how the ge&#173;ne&#173;tic makeup of an individual affects his or her response to drugs; in other words, the focus is on many genes.</p>
			<p class="glossary-entry"><span class="glossary-term">Phenocopy</span> The occurrence of a par&#173;tic&#173;u&#173;lar clinical phenotype as a result of &#173;either a nonmutagenic environmental &#173;factor (eg, exposure to a drug or virus) or an aty&#173;pi&#173;cal ge&#173;ne&#173;tic defect, when the more usually associated ge&#173;ne&#173;tic defect is absent.</p>
			<p class="glossary-entry"><span class="glossary-term">Phenotype</span> The observable or manifest physical, physiologic, biochemical, or molecular characteristics of an individual, &#173;either in &#173;whole or with regard to one or more traits, which are determined by the genotype but can be modified by the environment.</p>
			<p class="glossary-entry"><span class="glossary-term">Pleiotropism</span> Refers to multiple end effects (in dif&#173;fer&#173;ent organ systems) arising from a single mutant gene or gene pair.</p>
			<p class="glossary-entry"><span class="glossary-term">Polygenic inheritance</span> Inheritance determined by the operation of two or more genes. See <span class="glossary-entry_italic">Multifactorial inheritance.</span></p>
			<p class="glossary-entry"><span class="glossary-term">Polymerase chain reaction (PCR)</span> A technique by which segments of DNA or RNA can be amplified by use of flanking oligonucleotides called <span class="glossary-entry_italic">primers</span> and repeated cycles of amplification with DNA polymerase. The steps involve</p>
			<p class="glossary-bullet-list-mid ParaOverride-6">•&#9;heating to separate the molecules into single-&#173;stranded DNA</p>
			<p class="glossary-bullet-list-mid">•&#9;repeated annealing to the complementary target DNA sequences or primers specifically designed to delimit the beginning and ending of the target segment</p>
			<p class="glossary-bullet-list-mid">•&#9;extension of the primer sequences with the enzyme DNA polymerase, creating double-&#173;stranded DNA</p>
			<p class="glossary-bullet-list-mid ParaOverride-7">•&#9;separation of the products into single-&#173;stranded DNA</p>
			<p class="gloss-entry-cont">In effect, the amount of DNA is doubled with each cycle. Often, 30 or more cycles are used to obtain sufficient amplification for further testing.</p>
			<p class="glossary-entry-indent ParaOverride-8"><span class="glossary-entry_italic">Real-&#173;time PCR,</span> or <span class="glossary-entry_italic">quantitative PCR (qPCR),</span> is used to si&#173;mul&#173;ta&#173;neously amplify and quantify a targeted DNA molecule. <span class="glossary-entry_italic">Digital PCR (dPCR)</span> is a refinement used to directly quantify and clonally amplify nucleic acids. It is a more precise method than qPCR &#173;because it allows for more reliable collection and more sensitive mea&#173;sure&#173;ment of nucleic acid amounts. qPCR is useful for studying variations in gene sequences (eg, copy number variations and point mutations) and is routinely used to amplify samples for <span class="glossary-entry_italic">next-&#173;generation sequencing.</span></p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Polymorphism</span> The occurrence of two or more alleles at a specific locus with a frequency greater than 1% each in a given population.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Posttranslational modifications</span> Biochemical changes to or modifications of gene products &#173;after translation, including removal of amino acids from the end of the peptide, addition or removal of sugars, and addition of lipid side chains or phosphate groups to specific sites in the protein. Often, such changes are essential for proper protein localization or function.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Preimplantation ge&#173;ne&#173;tic diagnosis (PGD)</span> A ge&#173;ne&#173;tic test used with in vitro fertilization when one or both parents have a known ge&#173;ne&#173;tic abnormality. Testing is performed on cells removed from embryos in order to select one or more embryos that are &#173;free of the ge&#173;ne&#173;tic condition.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Prenatal diagnosis (PND)</span> Ge&#173;ne&#173;tic testing performed during pregnancy using amniocentesis, chorionic villus sampling (CVS), or cell-&#173;free fetal DNA (cffDNA) to determine &#173;whether a fetus carries a ge&#173;ne&#173;tic abnormality and allow parents to make reproductive choices.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Proband</span> The affected person whose disorder, or concern about a disorder, brings a &#173;family or pedigree to be genet&#173;ically evaluated. Also called the <span class="glossary-entry_italic">propositus</span> (male), <span class="glossary-entry_italic">proposita</span> (female), or <span class="glossary-entry_italic">index case.</span></p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Promoter</span> The sequence of nucleotides upstream (5<span class="glossary-entry_symbol">′</span>) from the coding sequence of a gene that determines the site of binding of RNA polymerase and, hence, initiation of transcription. Dif&#173;fer&#173;ent promoters for the same gene may exist and can result in alternately spliced gene products and tissue-&#173;specific expression. The promoter may contain the consensus DNA sequence (the so-&#173;called TATA box) approximately 25–30 bp (5<span class="glossary-entry_symbol">′</span>) upstream from the transcription start site.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Pseudodominance</span> The appearance of vertical transmission of a recessive ge&#173;ne&#173;tic disorder from 1 generation to the next due to an unusually high carrier frequency of a recessive allele and the resultant mating of an affected homozygote with an unaffected heterozygote, which results in 50% of offspring being affected. Pseudodominance implies recessive disease that has the appearance of dominant inheritance.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Pseudogene</span> A defective copy of a gene that often lacks introns and is rarely, if ever, expressed. Some pseudogenes are thought to arise by reverse transcription of mRNA that has had the introns spliced out. &#173;Others, such as globin pseudogenes, arise from silencing of a tandem duplicate. &#173;Because they are released from conservation (the maintenance of essential DNA sequences necessary for function) through se&#173;lection, pseudogenes often contain numerous base-&#173;pair changes and other mutational events (compared with the original functional gene).</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">q arm</span> The long arm of a chromosome. See <span class="glossary-entry_italic">p arm.</span></p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Recessive</span> Classically, describes a gene that results in a phenotype only in the homozygous or compound heterozygous state. See <span class="glossary-entry_italic">Dominant.</span></p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Recessive medical disorder</span> A disease state whose occurrence requires a homozygous (or compound heterozygous) genotype—&#173;that is, a double “dose” of the mutant allele. Heterozygotes are essentially clinically normal.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Recombinant</span> An individual with a combination of genes on a single chromosome unlike that in &#173;either parent. Usually applied to linkage analy&#173;sis, in which <span class="glossary-entry_italic">recombinant</span> refers to a haplotype (a set of alleles on a specific chromosome) that is not pre&#173;sent in &#173;either parent &#173;because of a recombination crossover.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Recombinant&#160;DNA</span> DNA that has been cut out of a single organism, reinserted into the DNA of a vector (plasmid or phage), and then reimplanted into a host cell. Also, any DNA that has been altered for further use.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Recombination</span> The formation of a new set of alleles on a single chromosome unlike that in &#173;either parent. Due to crossover during meiosis. See <span class="glossary-entry_italic">Crossing over.</span></p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Reference genomes</span> DNA sequences from control subjects in selected populations including the International HapMap and 1000 Genomes Proj&#173;ects.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Relatives, first-&#173;degree</span> Individuals who share on average half of their ge&#173;ne&#173;tic material with the proband: parents, siblings, offspring.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Relatives, second-&#173;degree</span> Individuals who share on average one-&#173;fourth of their ge&#173;ne&#173;tic material with the proband: grandparents, aunts and &#173;uncles, nieces and nephews, grandchildren.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Replication</span> Creation of a new linear DNA copy by the enzyme DNA polymerase, proceeding from the 5<span class="glossary-entry_symbol">′</span> side of bound primer to the 3<span class="glossary-entry_symbol">′</span> end of the DNA sequence. Replication of DNA occurs during chromosomal duplication.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Replication slippage</span> An error of DNA replication or copying. &#173;Because of the similarity of repeated base-&#173;pair sequences, one or more repeats may be skipped over and not represented in the copied DNA sequence.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Replicative segregation</span> The pro&#173;cess by which, through partitioning of copies of mitochondrial DNA (mtDNA) to each &#173;daughter cell during division, some cells receive a preponderance of normal or mutant copies. Replicative segregation tends to result in conversion of heteroplasmy to homoplasmy with associated development of disease within the affected tissue, if the tissue becomes homoplasmic for the mutant mtDNA. This phenomenon explains the development of new organ-&#173;system involvement in multisystem mitochondrial diseases.</p>
			<p class="glossary-entry"><span class="glossary-term">Reverse transcription</span> The pro&#173;cess, performed by the enzyme reverse transcriptase, in which mRNA is converted back to DNA. If the introns have already been spliced out of the precursor mRNA, the product of this pro&#173;cess is cDNA.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Sanger sequencing</span> The chain-&#173;termination method of sequencing DNA based on incorporating dideoxynucleotide molecules.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Segregation</span> The separation of pairs of alleles at meiosis.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Sex-&#173;linked</span> Refers to genes on the X or Y (sex) chromosome. The term is often used improperly to mean X-&#173;linked.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Short interfering RNA (siRNA)</span> A double-&#173;stranded RNA molecule (of 20–25 nucleotides) that plays a role in the RNA interference pathway, where it interferes with the expression of genes with complementary nucleotide sequences. Also known as <span class="glossary-entry_italic">small interference RNA.</span></p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Simplex</span> A term denoting that only 1 individual within a given &#173;family is affected by a condition known to have a heritable component. Thus, a single male or female with a ge&#173;ne&#173;tic disease is called a <span class="glossary-entry_italic">simplex case.</span> The term <span class="glossary-entry_italic">isolated</span> is also sometimes used.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Single nucleotide polymorphism (SNP)</span> Variation in a single base pair in a nucleotide sequence in the genome that occurs in more than 1% of the population. SNPs are rarely mutations that cause disease, are occasionally linked to disease-&#173;causing mutations, and usually are of unknown significance.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Single-&#173;nucleotide variant (SNV)</span> A single-&#173;nucleotide difference (substitution). The definition of SNV does not imply how often the variant occurs in a population.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Southern blotting</span> A method used in molecular biology for detection of a specific DNA sequence. Southern blotting combines transfer of electrophoresis-&#173;separated DNA fragments to a filter membrane and subsequent fragment detection by probe hybridization. Northern blotting is a similar pro&#173;cess performed with RNA and Western blotting with protein.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Splice junction site</span> The DNA region that demarcates the bound&#173;aries between exons and introns. The specific sequence determines &#173;whether the site acts as a 5<span class="glossary-entry_symbol">′</span> donor or a 3<span class="glossary-entry_symbol">′</span> acceptor site during splicing. Single-&#173;base-&#173;pair changes or mutations that involve splice junction sites may result in skipping of the following exon or incorporation of part of the adjacent intron into the mature mRNA.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Splicing</span> Pro&#173;cess by which the introns are removed from the precursor mRNA and the exons are joined together as mature mRNA prior to translation. Takes place within <span class="glossary-entry_italic">splice&#173;osomes,</span> specialized structures composed of RNA and proteins<span class="glossary-entry_italic">.</span></p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Sporadic</span> A trait that occurs in a single member of a kindred with no other &#173;family members affected. The term has been used by some ge&#173;ne&#173;ticists to imply that the trait is nonge&#173;ne&#173;tic.</p>
			<p class="glossary-entry ParaOverride-3"><span class="glossary-term">Stop codon (termination codon)</span> The DNA triplet that &#173;causes translation to end when the translation is coded into mRNA. The DNA stop codons are TAG, TAA, and TGA. Expressed as mRNA, &#173;these are UAG, UAA, and UGA.</p>
			<p class="glossary-entry"><span class="glossary-term">Telomeric&#160;DNA</span> A type of highly repetitive satellite DNA that forms the tips of chromosomes and prevents them from fraying or joining. It decreases in size as a consequence of the normal mechanisms of DNA replication in mitosis and may be impor&#173;tant in cellular senescence. Defects in the maintenance of telomeres may play a role in cancer formation.</p>
			<p class="glossary-entry ParaOverride-4"><span class="glossary-term">Threshold</span> In polygenic or multifactorial inheritance, a relatively sharp qualitative difference beyond which individuals are considered affected. The threshold is presumed to have been reached by the cumulative effects of the polygenic and multifactorial influences.</p>
			<p class="glossary-entry ParaOverride-4"><span class="glossary-term">Transcription</span> The synthesis, as catalyzed by a DNA-&#173;dependent RNA polymerase, of a single-&#173;stranded RNA molecule from the antisense strand of a double-&#173;stranded DNA template in the cell nucleus.</p>
			<p class="glossary-entry ParaOverride-4"><span class="glossary-term">Translation</span> The pro&#173;cess by which a polypeptide is synthesized from a sequence of specific mRNA.</p>
			<p class="glossary-entry ParaOverride-4"><span class="glossary-term">Translocation</span> The transfer of a part of 1 chromosome to a nonhomologous chromosome.</p>
			<p class="glossary-entry ParaOverride-4"><span class="glossary-term">Trinucleotide repeat expansion/contraction</span> The pro&#173;cess by which long sequences of multiple triplet codons (see <span class="glossary-entry_italic">Microsatellites</span>) are lengthened or shortened in the course of gene replication. The pro&#173;cess of expansion of trinucleotide repeats over consecutive generations results in the ge&#173;ne&#173;tic phenomenon of <span class="glossary-entry_italic">anticipation.</span> The under&#173;lying mechanisms for expansion (or contraction) appear to be replication slippage and unequal crossing over in the region of the repeats. Most disorders involving trinucleotide repeats are dominant in inheritance (eg, fragile X syndrome, myotonic dystrophy, Huntington disease, and Kennedy disease), but 1 is autosomal recessive (Friedreich ataxia). See <span class="glossary-entry_italic">Fragile sites.</span></p>
			<p class="glossary-entry ParaOverride-4"><span class="glossary-term">Tumor suppressor genes</span> Genes that must be pre&#173;sent in 1 fully functional copy to prevent uncontrolled cell proliferation. Two “hits” (mutations) of the gene, one for each allele, must occur in a given cell for tumor formation to occur. Examples include the genes for retinoblastoma, Wilms tumor, tuberous sclerosis, p53, ataxia-&#173;telangiectasia, and von Hippel–&#173;Lindau syndrome. Also called <span class="glossary-entry_italic">antioncogenes.</span> See <span class="glossary-entry_italic">Oncogene.</span></p>
			<p class="glossary-entry ParaOverride-4"><span class="glossary-term">Unequal crossing over</span> An error in the events of chromosomal duplication and cell division occurring during meiosis and, in rare cases, during mitosis. Prob&#173;ably &#173;because of similar sequences or repeated segments, chromosomal exchange occurs between nonhomologous regions of the chromosome, resulting in duplication and deletion of ge&#173;ne&#173;tic material in the &#173;daughter cells.</p>
			<p class="glossary-entry ParaOverride-4"><span class="glossary-term">Uniparental disomy</span> The conveyance to an offspring of 2 copies of an abnormal gene or chromosome by only 1 parent (the other parent makes no contribution). The child can be affected with autosomal recessive disease even if only 1 of the parents is a carrier of the abnormal gene. This occurrence has been reported in Stargardt disease, cystic fibrosis, Prader-&#173;Willi syndrome, and Angelman syndrome.</p>
			<p class="glossary-entry ParaOverride-4"><span class="glossary-term">Untranslated region (UTR)</span> The region upstream (5<span class="glossary-entry_symbol">′</span> UTR) or downstream (3<span class="glossary-entry_symbol">′</span> UTR) of the open reading frame (ORF) of a gene. The 5<span class="glossary-entry_symbol">′</span> UTR contains the promoter and part or all of the regulatory regions of the gene. The 3<span class="glossary-entry_symbol">′</span> UTR also serves impor&#173;tant functions in regulation and mRNA stability.</p>
			<p class="glossary-entry"><span class="glossary-term">Variant</span> A change in the DNA sequence. Historically, a variant that affects function or &#173;causes disease (pathogenic variant) is called a <span class="glossary-entry_italic">mutation</span>.</p>
			<p class="glossary-entry"><span class="glossary-term">Variant of unknown significance</span> A DNA change found in an individual that has not yet been reliably characterized as benign or pathogenic and/or whose functional consequences are uncertain.</p>
			<p class="glossary-entry"><span class="glossary-term">Vector</span> A viral, bacteriophage, or plasmid DNA molecule into which a stretch of genomic DNA or cDNA or a specific gene can be inserted. The <span class="glossary-entry_greek">λ</span>-&#173;bacteriophage can accept segments of DNA up to 25 kb long. Cosmid vectors can accommodate a segment 40 kb long. BAC (bacterial artificial chromosome) and YAC (yeast artificial chromosome) vectors can accept much larger fragments of DNA. Viral vectors such as <span class="glossary-entry_italic">adeno-&#173;associated virus (AAV)</span> have been used in gene therapy &#173;trials.</p>
			<p class="glossary-entry"><span class="glossary-term">Whole-&#173;exome sequencing (WES)</span> A laboratory technique for sequencing all the known protein-&#173;coding genes in an organism’s genome (known as the <span class="glossary-entry_italic">exome</span>).</p>
			<p class="glossary-entry"><span class="glossary-term">Whole-&#173;genome sequencing (WGS)</span> A laboratory pro&#173;cess to determine the complete DNA sequence of an organism’s genome.</p>
			<p class="glossary-entry"><span class="glossary-term">Wild type</span> A normal phenotype of an organism. Also, a normal allele as compared with a mutant allele.</p>
			<p class="glossary-entry"><span class="glossary-term">X-&#173;linked</span> Refers to genes on the X chromosome.</p>
			<p class="glossary-entry"><span class="glossary-term">Y-&#173;linked</span> Refers to genes on the Y chromosome.</p>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer001" class="Basic-Text-Frame">
			</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer002" class="Basic-Text-Frame">
			</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer003" class="Basic-Text-Frame">
			</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer004" class="Basic-Text-Frame">
			</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer005" class="Basic-Text-Frame">
			</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer006" class="Basic-Text-Frame">
			</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer007" class="Basic-Text-Frame">
			</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer008" class="Basic-Text-Frame">
			</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer009" class="Basic-Text-Frame">
			</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer010" class="Basic-Text-Frame">
			</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer011" class="Basic-Text-Frame">
			</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer012" class="Basic-Text-Frame">
			</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer013" class="Basic-Text-Frame">
			</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer014" class="Basic-Text-Frame">
			</div>
		</div>
	</body>
</html>
