<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
	<head>
		<meta charset="utf-8" />
		<title>BCSC2019_S02_C05_p169_198_9P</title>
		<link href="BCSC2019_S02_C05_p169_198_9P-web-resources/css/idGeneratedStyles.css" rel="stylesheet" type="text/css" />
	</head>
	<body id="BCSC2019_S02_C05_p169_198_9P" lang="en-US">
		<div id="_idContainer000" class="Basic-Text-Frame">
			<p class="part-title-USE">Genetics</p>
		</div>
		<div id="_idContainer001" class="Basic-Text-Frame">
			<p class="Year---Cover">Part III</p>
		</div>
		<div id="_idContainer002" class="Basic-Text-Frame">
            <p class="chapter-title">Molecular  Genetics</p>
            <p class="h1 ParaOverride-1">Introduction</p>
			<p class="body-text--no-indent-">Ge&#173;ne&#173;tics is the study of heredity and the variations in inherited characteristics and diseases. Although ge&#173;ne&#173;tics is a relatively new science compared with such disciplines as anatomy and physiology, its significance in the overall understanding of &#173;human life cannot be overstated. Ge&#173;ne&#173;tic knowledge can enhance our understanding of the pro&#173;cesses of cellular function, embryology, and development, as well as our concepts of disease. Many researchers think that as much as 90% of medical disease &#173;either has a major ge&#173;ne&#173;tic component or involves ge&#173;ne&#173;tic &#173;factors that may significantly influence the disease.</p>
			<p class="body-text">The discovery of previously unknown genes, such as the <span class="italic">homeobox genes</span> (eg, the <span class="italic">HOX</span> and <span class="italic">PAX</span> gene families)—&#173;which regulate, guide, and coordinate early embryologic development and differentiation—&#173;has opened new areas of understanding of physiology at the cellular or tissue level. (&#173;These genes, also called <span class="italic">homeotic selector genes,</span> are discussed in <span class="xref-local">Part II, Embryology, as well</span>.) Another example is the identification of the genes that appear to be transcribed as initiating events in the pro&#173;cess of <span class="italic">apoptosis,</span> or programmed cell death, which itself appears crucial for normal embryogenesis as well as for degenerative diseases and cancers.</p>
			<p class="body-text">Ge&#173;ne&#173;tic disorders affect about 5% of live-&#173;born infants in the United States. Approximately 50% of childhood blindness has a ge&#173;ne&#173;tic cause. Some 20,000–25,000 &#173;human genes involving about 180,000 exons are known. In 10%–15% of known ge&#173;ne&#173;tic diseases, clinical findings are limited to the eye; a similar percentage includes systemic disorders with ocular manifestations.</p>
			<p class="h1">Terminology</p>
			<div id="Chapt5_Top1">
			<p class="body-text--no-indent-">Familiarity with the vocabulary of ge&#173;ne&#173;tics and molecular biology &#173;will greatly enhance the reader’s understanding of the following 2 chapters on molecular and clinical ge&#173;ne&#173;tics. The reader is thus encouraged to review the ge&#173;ne&#173;tics glossary in the appendix of this book, which includes many key ge&#173;ne&#173;tics terms, as well as to consult online resources, 2 examples of which follow.</p>
			<p class="reference--journal--first">National Cancer Institute Dictionary of Ge&#173;ne&#173;tics Terms. <a href="https://www.cancer.gov/publications/dictionaries/genetics-dictionary"><span class="Hyperlink">https://&#173;www.&#173;cancer.&#173;gov/&#173;publications/&#173;dictionaries/&#173;genetics-&#173;dictionary</span></a></p>
			<p class="reference--journal--last">National &#173;Human Genome Research Institute Talking Glossary of Ge&#173;ne&#173;tic Terms. <a href="https://www.genome.gov/glossary/"><span class="Hyperlink">https://&#173;www.&#173;genome.&#173;gov/&#173;glossary/</span></a></p>
			<p class="chapter-number ParaOverride-1"><span class="cn-chap">Chapter</span><span class="cn-chap"> </span>5</p>
			<p class="chapter-title">Molecular Ge&#173;ne&#173;tics</p>
			</div>
			<p class="h1 ParaOverride-2">Highlights</p>
			<div id="Chapt5_Top2">
			<p class="body-text--no-indent-">
			<ul>
				<li class="bullet-list-first ParaOverride-3">Cell division occurs via a complex pro&#173;cess known as the <span class="bullet-list_italic">cell cycle.</span> The function of tumor suppressor genes is to regulate this cycle. Mutations to &#173;these genes result in numerous conditions with ophthalmic manifestations, examples of which are neurocutaneous disorders (phakomatoses) and retinoblastoma.</li>
				<li class="bullet-list-mid">Approximately 95% of DNA does not code for proteins and may be involved in regulation of gene expression. The term <span class="bullet-list_italic">epigenet&#173;ics</span> refers to the study of heritable pro&#173;cesses that alter gene expression without changing the DNA sequence.</li>
				<li class="bullet-list-mid">Transcription &#173;factors determine the rate of messenger RNA production by DNA. The &#173;family of <span class="bullet-list_italic">PAX</span> genes encodes for transcription &#173;factors, mutations of which are involved in the development of numerous ophthalmic conditions.</li>
				<li class="bullet-list-mid">Alternate splicing allows dif&#173;fer&#173;ent isoforms of a par&#173;tic&#173;u&#173;lar protein to be expressed. Vascular endothelial growth &#173;factor and its receptors have vari&#173;ous isoforms due to this mechanism.</li>
				<li class="bullet-list-mid">Mitochondrial DNA (mtDNA) is passed on to &#173;children from their &#173;mothers. Many diseases with ophthalmic manifestations occur &#173;because of mutations in mtDNA, including chronic progressive external ophthalmoplegia and Leber hereditary optic neuropathy.</li>
				<li class="bullet-list-last ParaOverride-4">New gene therapies such as AAV (adeno-&#173;associated virus) vector gene therapy and the CRISPR-&#173;Cas9 system have the potential to treat many previously untreatable eye diseases.</li>
			</ul>
			</p>
			</div>
			<p class="h1 ParaOverride-5">The Cell Cycle</p>
			<div id="Chapt5_Top3">
			<p class="body-text--no-indent-">The cell cycle is the series of events that take place in a cell leading to its duplication and division (<span class="xref-figure">Fig 5-1</span>). The 4 distinct phases are</p>
			<ul>
				<li class="bullet-list-first ParaOverride-6">G<span class="bullet-list_subscript _idGenCharOverride-1">1</span> (growth, preparation for DNA synthesis)</li>
				<li class="bullet-list-mid">S (DNA synthesis/chromosome replication)</li>
				<li class="bullet-list-mid">G<span class="bullet-list_subscript _idGenCharOverride-1">2</span> (growth, preparation for mitosis)</li>
				<li class="bullet-list-last ParaOverride-7">M (mitosis and cytokinesis)</li>
			</ul>
			<p class="body-text--no-indent-">The M phase consists of 2 pro&#173;cesses: <span class="italic">mitosis,</span> in which the cell’s chromosomes are divided between the 2 &#173;sister cells, followed by <span class="italic">cytokinesis,</span> in which the cell’s cytoplasm divides in half and forms distinct cells. Cells that have temporarily stopped dividing are said to have entered a state of quiescence called the <span class="italic">G</span><span class="italic_subscript _idGenCharOverride-1">0</span> <span class="italic">phase.</span> The M phase can be subdivided into several distinct, sequential phases:</p>
			<ul>
				<li class="bullet-list-first"><span class="bullet-list_italic">prophase</span> (chromatin is condensed into chromosomes)</li>
				<li class="bullet-list-mid"><span class="bullet-list_italic">metaphase</span> (chromosomes align in the &#173;middle of the cell)</li>
				<li class="bullet-list-mid"><span class="bullet-list_italic">anaphase</span> (chromosomes split and migrate to opposite poles of the cell)</li>
				<li class="bullet-list-last"><span class="bullet-list_italic">telophase</span> (2 &#173;daughter nuclei form at the poles of the cell)</li>
			</ul>
			<p class="body-text--no-indent-">Mitosis refers to somatic cell division, whereas meiosis refers to replication of germ cells.</p>
			<p class="h2">Meiosis</p>
			<p class="body-text--no-indent-"><span class="italic">Meiosis</span> is a specialized type of cell division necessary for sexual reproduction in eukaryotes &#173;because the cells produced by meiosis are ova and sperm. It consists of 2 successive cell divisions, meiosis I and meiosis II. Unlike in mitosis, in meiosis the chromosomes undergo a recombination that shuffles the genes from each parent, producing a dif&#173;fer&#173;ent ge&#173;ne&#173;tic combination in each gamete. The outcome of meiosis is 4 genet&#173;ically unique haploid cells, whereas the outcome of mitosis is 2 genet&#173;ically identical diploid cells.</p>
			<p class="body-text">Interphase consists of the G<span class="subscript _idGenCharOverride-1">1</span> and S phases (&#173;there is no G<span class="subscript _idGenCharOverride-1">2</span> phase in meiosis) and is followed by meiosis I and then meiosis II. Meiosis I and II are each divided into prophase, metaphase, anaphase, and telophase stages, as in the mitotic cell cycle. In the G<span class="subscript _idGenCharOverride-1">1</span> phase, each of the chromosomes consists of a single (very long) molecule of DNA. At this stage in &#173;humans, the cells contain 46 chromosomes, the same number as in somatic cells. During S phase, the chromosomes duplicate, so that each of the 46 chromosomes becomes a complex of 2 identical &#173;sister chromatids.</p>
			<p class="body-text">During meiosis I, homologous chromosomes (a matched pair, 1 derived from each parent) separate into 2 cells. The entire haploid content of each chromosome is contained in each of the resulting &#173;daughter cells; the first meiotic division thus reduces the ploidy of the original cell by half.</p>
			<p class="body-text">During meiosis II, each chromosome’s &#173;sister strands (the chromatids) are decoupled, and the individual chromatids are segregated into haploid &#173;daughter cells. The 2 cells resulting from meiosis I divide during meiosis II, creating 4 haploid &#173;daughter cells.</p>
			<p class="body-text">Chromosomal <span class="italic">crossing over</span> is the exchange of ge&#173;ne&#173;tic material between homologous chromosomes that results in recombinant chromosomes. It occurs during prophase of the first meiotic division (prophase I), usually when matching regions on matching chromosomes break and then reconnect to the other chromosome. Although the same genes appear in the same order, the alleles are dif&#173;fer&#173;ent. It is theoretically pos&#173;si&#173;ble to have any combination of parental alleles in an offspring. This theory of <span class="italic">in&#173;de&#173;pen&#173;dent assortment</span> of alleles is fundamental to ge&#173;ne&#173;tic inheritance. However, the chances of recombination are greater the farther apart 2 genes are from each other. The ge&#173;ne&#173;tic distance is described in centimorgans (cM; named for Thomas Hunt Morgan, who described crossing over), and a distance of 1 cM between genes represents a 1% chance of their crossing over in 1 meiosis.</p>
			<p class="body-text">Ge&#173;ne&#173;tic <span class="italic">linkage</span> describes the tendency of genes to be inherited together as a result of their proximity on the same chromosome. <span class="italic">Linkage disequilibrium</span> occurs when combinations of alleles are pres&#173;ent in a population more or less frequently than would be expected based on their distances apart from each other. This concept is applied in searches for a gene that may cause a par&#173;tic&#173;u&#173;lar disease.</p>
			<p class="body-text">Although crossovers typically occur between homologous regions of matching chromosomes, a mismatch or unbalanced recombination may occur. This rare event can be a local duplication or deletion of genes on 1 chromosome, a translocation of part of 1 chromosome onto a dif&#173;fer&#173;ent one, or an inversion of a part of the chromosome.</p>
			<p class="h2 ParaOverride-8">Cell Cycle Regulation</p>
			<p class="body-text--no-indent-">In the cell cycle, transition from one phase to the next is regulated at checkpoints (see Fig 5-1). Impor&#173;tant checkpoints occur at the following:</p>
			<ul>
				<li class="bullet-list-first ParaOverride-6">G<span class="bullet-list_subscript _idGenCharOverride-1">1</span>: transition from G<span class="bullet-list_subscript _idGenCharOverride-1">1</span> to S</li>
				<li class="bullet-list-last ParaOverride-7">G<span class="bullet-list_subscript _idGenCharOverride-1">2</span>: transition from G<span class="bullet-list_subscript _idGenCharOverride-1">2</span> to M</li>
			</ul>
			<p class="body-text">Checkpoints allow monitoring of the cell to verify the successful, error-&#173;free completion of the previous phase. At the G<span class="subscript _idGenCharOverride-1">1</span> checkpoint, cell size and the availability of nutrients and growth &#173;factors are assessed, and the cell is checked for DNA damage. &#173;After completion of this checkpoint, the cell is committed to proceeding with cell division; other&#173;wise, it enters the quiescent G<span class="subscript _idGenCharOverride-1">0</span> phase. Before the cycle progresses to the M phase, further inspection of the DNA occurs at the G<span class="subscript _idGenCharOverride-1">2</span> checkpoint. If damaged DNA is detected at &#173;either checkpoint, it may be repaired, or programmed cell death (see the section Apoptosis) may be initiated.</p>
			<p class="body-text">Checkpoint regulation occurs via a &#173;family of proteins known as <span class="italic">cyclins</span> and <span class="italic">cyclin-&#173;</span><span class="italic">dependent kinases (CDKs).</span> At the G<span class="subscript CharOverride-1">1</span> checkpoint, CDK phosphorylation of proteins of the retinoblastoma (Rb) &#173;family facilitates downstream transcription in preparation for S phase. Tumor suppressor genes like the Rb &#173;family often have a role in regulation of the cell cycle, dysregulation of which can lead to cancer (see the section “Tumor suppressor genes”).</p>
			<p class="reference--journal--single ParaOverride-9">Sun A, Bagella L, Tutton S, Romano G, Giordano&#160;A. From G0 to S phase: a view of the roles played by the retinoblastoma (Rb) &#173;family members in the Rb-&#173;E2F pathway. <span class="reference--journal-_italic">J Cell Biochem.</span> 2007;102(6):1400–1404.</p>
			</div>
			<p class="h1 ParaOverride-10">Gene Structure</p>
			<div id="Chapt5_Top4">
			<p class="body-text--no-indent-">Composed of DNA, <span class="italic">genes</span> are the molecular units of heredity and are located primarily in the cell nucleus, where they are assembled into <span class="italic">chromosomes</span> of varying sizes. Paired chromosomes are numbered from largest (1) to smallest (22), and &#173;there are 2 additional sex chromosomes (XY or XX). The 4 bases pres&#173;ent in DNA—&#173;adenine (A), cytosine (C), guanine (G), and thymine (T)—&#173;are combined into a double-&#173;helix structure that allows replication, transcription, and translation. The ge&#173;ne&#173;tic structure (<span class="xref-figure">Fig 5-2</span>) can be likened to the sections of an encyclopedia, with genes the chapters, <span class="italic">exons</span> the sentences, <span class="italic">trinucleotides</span> the words, and <span class="italic">nucleotides</span> the letters.</p>
			<p class="body-text"><span class="italic">Mitochondria,</span> the site of oxidative phosphorylation, are the power plants of the cell. The mitochondria are a vestige of a symbiotic relationship between 2 primitive unicellular organisms that merged to form eukaryotic organisms (most animals and plants). The fact that mitochondria still contain their own DNA is a reminder of their in&#173;de&#173;pen&#173;dent origin. Each mitochondrion contains 2–10 copies of a very short, circular segment containing 13 protein-&#173;coding genes involved in oxidative phosphorylation. &#173;Because mitochondria contain several segments of DNA and each cell contains several mitochondria, &#173;there may be variation of the mitochondrial DNA (mtDNA) within a cell and between cells of the same person, a state known as <span class="italic">heteroplasmy.</span> &#173;Humans acquire mitochondria from the ovum, and thus mtDNA follows maternal line inheritance.</p>
			<p class="body-text">Chromosomal DNA replication and RNA synthesis (transcription) occur within the nucleus. Messenger RNA (mRNA) is transported to ribosomes in the cytoplasm, where translation to the amino acid sequences of proteins occurs. Following the mRNA molecule’s initiation codon (start sequence) is the structural <span class="italic">open reading frame (ORF),</span> which is composed of <span class="italic">exons</span> (sequences that code for amino acids that &#173;will be pres&#173;ent in the final protein) and <span class="italic">introns</span> (sequences that are spliced out during the pro&#173;cessing of mRNA). Following the last exon is the <span class="italic">3&apos;</span> <span class="italic">untranslated region (3&apos;</span> <span class="italic">UTR).</span> The function of this region is partly regulatory.</p>
			<p class="body-text">The development of introns in higher organisms may have had evolutionary benefits. The compartmentalization of coding segments into exons may have permitted more rapid evolution of proteins by allowing for alternative pro&#173;cessing of precursor RNA (alternative splicing) and for rearrangements of exons during gene duplication (exon shuffling). Some introns contain complete, separate genes, and some of &#173;these may cause disease or influence the expression of other genes. Expansion of unstable repeats within introns can cause abnormal splicing and result in ge&#173;ne&#173;tic disease. Small insertions and deletions are very common and referred to as <span class="italic">indels.</span></p>
			</div>
			<p class="h1">Noncoding DNA</p>
			<div id="Chapt5_Top5">
			<p class="body-text--no-indent-">The majority of DNA—&#173;approximately 95% of the base sequences in &#173;human DNA—&#173;does not code for proteins. Noncoding DNA is composed of highly repetitive sequences, some of which include <span class="italic">satellites, microsatellites, short interspersed ele&#173;ments (SINEs),</span> and <span class="italic">long interspersed ele&#173;ments (LINEs).</span> The 300-&#173;base-&#173;pair (bp) <span class="italic">Alu</span> sequence, named &#173;after the restriction enzyme used to identify it, is the repetitive DNA that appears most frequently. Noncoding DNA comprises introns, promoters, and other regions within chromosomes and mitochondria and is involved in regulating gene expression and exon splicing.</p>
			<p class="body-text">RNA transcribed from noncoding DNA may directly influence the transcription of other sequences and participate in normal genome repair and regulation. Some of the repetitive sequences of nontranscribed DNA form <span class="italic">telomeric DNA,</span> which is essential for the correct formation and maintenance of chromosomes. Loss of telomeric DNA correlates with cell senescence. Defects in telomeric DNA maintenance have been proposed to be associated with carcinogenesis.</p>
			</div>
			<p class="h1">Gene Transcription and Translation: The Central Dogma of Genetics</p>
			<div id="Chapt5_Top6">
			<p class="body-text--no-indent-">The central dogma of gene transcription and translation is that the DNA code is transcribed as mRNA code and then translated as amino acid code of the resulting protein (<span class="xref-figure">Fig 5-3</span>). The trinucleotides that correspond with amino acids have some redundancy in the system, so that a nucleotide change may not necessarily result in a change in amino acid. The coding region of DNA is composed of exons, several of which are spliced together to make the full coding sequence of RNA. Although the central dogma specifies that DNA determines RNA sequence and that RNA determines amino acid sequence, &#173;there are feedback and regulatory mechanisms of gene expression that are both genet&#173;ically and environmentally determined. &#173;These mechanisms, such as methylation and histone formation, can silence gene expression. In addition, small segments of RNA can block mRNA. The study of the influence of &#173;these regulatory mechanisms in gene and disease expression is known as <span class="italic">epigenet&#173;ics.</span></p>
			<p class="body-text">Genes control cellular activity through 2 pro&#173;cesses:</p>
			<p class="numbered-list-first ParaOverride-11">1.&#9;transcription <span class="numbered-list_italic">(expression),</span> in which DNA molecules give rise to RNA molecules, followed by translation in most cases</p>
			<p class="numbered-list-last ParaOverride-12">2.&#9;<span class="numbered-list_italic">translation,</span> in which RNA directs the synthesis of proteins. Translation occurs in ribosomes, where mRNA induces transfer RNA (tRNA)–&#173;mediated recruitment of&#160;amino acids to “build” a protein. A more in-&#173;depth description of translation is beyond the scope of this chapter.</p>
			<p class="body-text"><span class="italic">Transcription &#173;factors</span> are proteins that bind to specific DNA sequences and thus control the flow (or transcription) of ge&#173;ne&#173;tic information from DNA to mRNA. Transcription &#173;factors perform this function by promoting or repressing the recruitment of RNA polymerase to specific genes.</p>
			<p class="body-text">Approximately 10% of genes in the &#173;human genome code for transcription &#173;factors. They contain one or more DNA-&#173;binding domains, which attach to specific sequences of DNA adjacent to the genes that they regulate. &#173;There are numerous families of &#173;these genes, including the homeobox and paired box genes. <span class="italic">PAX6</span> acts as a master control gene for the development of the eye, an example of the key role of transcription &#173;factors in embryogenesis.</p>
			<p class="body-text">Many ophthalmic diseases result from transcription-&#173;factor mutations. <span class="italic">PAX2</span> mutations cause colobomas of the optic nerve and renal hypoplasia. <span class="italic">PAX3</span> mutations cause Waardenburg syndrome with dystopia canthorum (types WS1 and WS3). <span class="italic">PAX6</span> mutations are the basis of virtually all cases of aniridia, occasional cases of Peters anomaly, and several other rarer phenotypes, specifically autosomal dominant keratitis and dominant foveal hypoplasia.</p>
			<p class="reference--journal--single ParaOverride-9">Fitzpatrick DR, van Heyningen&#160;V. Developmental eye disorders. <span class="reference--journal-_italic">Curr Opin Genet Dev.</span> 2005;15(3):348–353.</p>
			<p class="h2 ParaOverride-8">Intron Excision</p>
			<p class="body-text--no-indent-">Messenger RNA undergoes excision of the introns by a highly or&#173;ga&#173;nized pro&#173;cess called <span class="italic">splicing,</span> which leaves the mRNA composed of only exons, or coding segments. The exons can then undergo translation in the ribosomes. Splicing takes place in specialized structures called <span class="italic">spliceosomes,</span> which are composed of RNA and proteins. Errors of splicing can lead to ge&#173;ne&#173;tic disease. Approximately 15% of point mutations that cause &#173;human disease do so by generating splicing errors that result in aberrations such as exon skipping, intron retention, or use of a cryptic splice site. For example, mutations in proteins that are vital in splicing can cause retinitis pigmentosa (RP).</p>
			<p class="h2 ParaOverride-8">Alternative Splicing and Isoforms</p>
			<p class="body-text--no-indent-">Alternative splicing is the creation of multiple pre-&#173;mRNA sequences from the same gene by the action of dif&#173;fer&#173;ent promoters. &#173;These promoters cause certain exons to be skipped during transcription of the gene. The protein products of alternative splicing are often called <span class="italic">isoforms.</span> The promoters are usually tissue specific, so dif&#173;fer&#173;ent tissues express dif&#173;fer&#173;ent isoforms. The gene for dystrophin is an example of alternative splicing: full-&#173;length dystrophin is the major isoform expressed in muscle; shorter isoforms predominate in the ret&#173;ina, peripheral nerve, and central ner&#173;vous system.</p>
			<p class="body-text">Another example of alternative splicing’s relevance underlies the basis of the cornea’s avascularity. Vascular endothelial growth &#173;factor (VEGF) receptor 1 is a key blood vessel receptor that binds and transduces a signal from the primary mediator of angiogenesis, VEGF. In the cornea, high levels of an alternatively spliced isoform, soluble VEGF receptor 1 (sVEGFR-1), are expressed. As this isoform is soluble, it is pres&#173;ent in the extracellular matrix and serves as an endogenous VEGF trap or decoy receptor. Without it, &#173;there are increased levels of &#173;free VEGF, and the cornea becomes vulnerable to vascular invasion.</p>
			<p class="reference--journal--single ParaOverride-9">Ambati BK, Nozaki M, Singh N, et&#160;al. Corneal avascularity is due to soluble VEGF receptor-1. <span class="reference--journal-_italic">Nature.</span> 2006;443(7114):993–997.</p>
			<p class="h2 ParaOverride-13">Methylation</p>
			<p class="body-text--no-indent-">Regions of DNA that are undergoing transcription lack 5-&#173;methyl cytosine residues, which normally account for 1%–5% of total DNA. Evidence suggests a close correlation between methylation and gene inactivation. Regulation of DNA methylation may be responsible for imprinting control. Methylation may account for variation in phenotypic expression of some diseases.</p>
			<p class="reference--journal--single ParaOverride-9">Hjelmeland LM. Dark &#173;matters in AMD ge&#173;ne&#173;tics: epigenet&#173;ics and stochasticity. <span class="reference--journal-_italic">Invest Oph</span><span class="reference--journal-_italic">thalmol Vis Sci.</span> 2011;52(3):1622–1631.</p>
			<p class="h2 ParaOverride-13">X-&#173;Inactivation</p>
			<p class="body-text--no-indent-">The random inactivation of 1 of the 2 X chromosomes in the female, resulting in the lack of expression of the majority of genes on that chromosome, is a significant event during early development of the &#173;human embryo. The time of X-&#173;inactivation is not precisely known but is thought to vary over a period of several cell divisions during the blastocyst–&#173;gastrula transition. X-&#173;inactivation is also known as <span class="italic">lyonization,</span> &#173;after its discoverer, Mary Lyon. Lyonization affects the severity of the phenotype of several X-&#173;linked ret&#173;i&#173;nal conditions, such as RP and incontinentia pigmenti.</p>
			<p class="h2 ParaOverride-13">Imprinting</p>
			<p class="body-text--no-indent-">Genomic imprinting is a heritable yet reversible pro&#173;cess by which a gene is modified, depending on which parent provides it. The mechanism is unclear but appears to operate at the chromatin organ&#173;ization level and involves heterochromatization and methylation of CpG (cytosine-&#173;phosphate-&#173;guanine) sites. Examples of genes that can be imprinted include the Wilms tumor–&#173;suppressor gene and the &#173;human <span class="italic">SNRPN</span> (small nuclear ribonucleoprotein polypeptide N) gene.</p>
			<p class="body-text">Prader-&#173;Willi and Angelman syndromes are examples of diseases resulting from abnormalities of imprinting. Approximately 70%–80% of patients with Prader-&#173;Willi syndrome harbor a deletion of the paternally derived 15q11–&#173;q13, resulting in the loss of this region’s normal contribution from the paternal line. About 70%–80% of patients with Angelman syndrome also have a deletion of 15q11–&#173;q13, but from the maternally derived chromosome, resulting in loss of the maternal contribution. Chromosome 15 uniparental disomy, wherein both copies of chromosome 15 are inherited from the same parent, can also cause each syndrome. The 2 chromosomes 15&#160;in uniparental disomy are maternal in Prader-&#173;Willi syndrome and paternal in Angelman syndrome. The <span class="italic">SNRPN</span> gene maps to 15q11–&#173;q13 but appears to be expressed only from the paternally inherited allele.</p>
			</div>
			<p class="h1 ParaOverride-14">DNA Damage and Repair</p>
			<div id="Chapt5_Top7">
			<p class="body-text--no-indent-">DNA is constantly sustaining damage from mutagens such as ultraviolet (UV) light, chemicals, and spontaneous deamination. Each cell loses 10,000 bases per day from spontaneous DNA breakdown related to normal body temperature alone. In the absence of repair, &#173;these mutations would accumulate and result in tumor formation. Damaged DNA is estimated to cause approximately 80%–90% of cancers in &#173;humans.</p>
			<p class="h2 ParaOverride-15">Repair</p>
			<p class="body-text--no-indent-">Damaged DNA sites are repaired chiefly by 2 mechanisms: <span class="italic">excision repair</span> and <span class="italic">mismatch repair.</span> The pro&#173;cesses of replication, transcription, mismatch repair, excision repair, and gene expression are closely coordinated by cross-&#173;acting systems. Enzymes that cut or patch segments of DNA during crossing over at meiosis are also involved in DNA repair. Molecules that unwind double-&#173;stranded DNA (called <span class="italic">helicases</span>) are involved in replication, transcription, and DNA excision repair.</p>
			<p class="body-text">The <span class="italic">antioncogene p53</span> appears to play an extremely impor&#173;tant role as the “guardian of the genome” by preventing cells from proliferating if their DNA is irreparably damaged. Levels of p53 increase &#173;after UV or ionizing radiation exposure. The p53 gene inhibits DNA replication directly and binds with 1 of the RNA polymerase transcription &#173;factors, TFIIH. If the degree of damage is slight, increased production of p53 induces reversible cell arrest &#173;until DNA repair can take place. If DNA damage is too &#173;great or irreversible, p53 production is massively increased and apoptosis occurs, prob&#173;ably through stimulation of the expression of the <span class="italic">BAX</span> gene, whose product promotes apoptosis. Loss of p53 &#173;causes cells to fail to arrest in response to DNA damage, and &#173;these cells do not enter apoptosis. Thus, mutations of p53 predispose to tumorigenesis.</p>
			<p class="body-text">The gene mutated in ataxia-&#173;telangiectasia (Louis-&#173;Bar syndrome), a protein kinase called <span class="italic">ATM,</span> also appears to be integrally involved in DNA repair, possibly by informing the cell of radiation damage. The <span class="italic">ATM</span> gene product associates with synaptonemal complexes, promotes chromosomal synapsis, and is required for meiosis. Individuals with ataxia-&#173;telangiectasia have a threefold greater risk of cancer.</p>
			<p class="body-text">Xeroderma pigmentosum is a severe condition in which the functions of enzymes that repair UV-&#173;damaged DNA are crippled. Patients with this condition typically have diffuse pigmented anomalies on their sun-&#173;exposed skin and are at high risk for basal cell and squamous cell carcinoma, as well as melanoma. Ocular surface cancers (squamous cell carcinoma and melanoma) can also develop in affected patients.</p>
			<p class="reference--journal--single ParaOverride-9">Lim R, Sethi M, Morley AMS. Ophthalmic manifestations of xeroderma pigmentosum: a perspective from the United Kingdom. <span class="reference--journal-_italic">Ophthalmology</span>. 2017;124(11):1652–1661.</p>
			<p class="h2 ParaOverride-15">Apoptosis</p>
			<p class="body-text--no-indent-"><span class="italic">Apoptosis</span> is a Greek word describing leaves dropping from trees. (<span class="italic">Ptosis,</span> drooping of the upper eyelid, comes from the same root.) Apoptosis is the pro&#173;cess of programmed cell death that occurs in multicellular organisms, in contrast to necrosis, a form of traumatic cell death that results from acute cellular injury. Biochemical events in apoptosis result in characteristic cell changes and cell death. Morphological changes include cell shrinkage, nuclear fragmentation, chromatin condensation, and chromosomal DNA fragmentation. Several key events in apoptosis focus on the mitochondria, including the release of caspase activators, changes in electron transport, loss of mitochondrial transmembrane potential, altered cellular reduction-&#173;oxidation (redox) reactions, and activation of pro-&#173; and antiapoptotic Bcl-2 &#173;family proteins.</p>
			<p class="body-text">Apoptosis is crucial in the developing &#173;human embryo; scaffolding cells such as &#173;those involved in eyelid opening are removed by epidermal apoptosis. In &#173;later life, excessive apoptosis &#173;causes atrophy, such as occurs in RP or glaucoma, whereas insufficient apoptosis results in uncontrolled cell proliferation, such as occurs in cancers, including retinoblastoma.</p>
			</div>
			<p class="h1 ParaOverride-14">Mutations and Disease</p>
			<div id="Chapt5_Top8">
			<p class="h2-h1">Mutations Versus Polymorphisms</p>
			<p class="body-text--no-indent-">Mutations are changes in DNA that can lead to disease, whereas polymorphisms are variations in DNA that &#173;were previously thought to rarely cause disease. The difference is not always easy to determine. In general, mutations change amino acid sequence or, more dramatically, lead to a shortening or nonproduction of the protein encoded by the gene. Polymorphisms tend not to cause a change in the amino acid sequence (&#173;because of the built-in redundancy in the DNA code) or a change from one amino acid to a similar amino acid. However, some synonymous changes, though not changes in amino acid sequence, could affect splicing. Many of the disease-&#173;associated single nucleotide polymorphisms (SNPs) identified in genome-&#173;wide association studies (GWAS) are found in the noncoding regions of the genome.</p>
			<p class="h3 ParaOverride-16">Mutations</p>
			<p class="body-text--no-indent-">Mutations can involve a change in a single base pair; &#173;simple deletion or insertion of DNA material; or more complex rearrangements such as inversions, duplications, or translocations. Deletion, insertion, or duplication of any number of base pairs in other than groups of 3 creates a frameshift in the entire DNA sequence downstream, resulting in the eventual formation of a stop codon and truncation of the message.</p>
			<p class="body-text">Mutations that result in no active gene product being produced are called <span class="italic">null mutations.</span> Null mutations include missense or nonsense mutations that (1) produce &#173;either a stop codon directly or a frameshift with creation of a premature stop codon downstream or (2) cause an alteration at the acceptor splice junction site, resulting in the loss of exons or inappropriate incorporation of introns into the spliced mRNA.</p>
			<p class="body-text">Mutations can also lead to a gain of function that may be beneficial (leading to evolution) or detrimental (leading to disease). An example of a beneficial gain in function is the emergence, among bacteria, of antibiotic re&#173;sis&#173;tance. An example of a detrimental gain of function is a receptor protein that binds too tightly with its target protein, creating loss of normal physiologic function. Most autosomal dominant disorders are of this type.</p>
			<p class="body-text">Single base-&#173;pair mutations may code for the same amino acid or a tolerable change in the amino acid sequence, leading to harmless polymorphisms or DNA variations that are in turn inherited. &#173;These are called <span class="italic">conserved base-&#173;pair mutations.</span></p>
			<p class="h3 ParaOverride-16">Polymorphisms</p>
			<p class="body-text--no-indent-">A polymorphism is any variation in DNA sequence that occurs, by convention, at a frequency of greater than 1% in the normal population. Key polymorphisms associated with disease include &#173;those in the region of the <span class="italic">CFH</span> gene in age-&#173;related macular degeneration and the <span class="italic">LOXL1</span> gene in pseudoexfoliation syndrome. Many polymorphisms are &#173;silent and simply linked to the disease mutation, but some may influence disease.</p>
			<p class="h2 ParaOverride-15">Cancer Genes</p>
			<p class="body-text--no-indent-">Cancer can result from any of a number of ge&#173;ne&#173;tic mechanisms, including the activation of oncogenes and the loss of tumor suppressor genes. The product of proto-&#173;oncogenes is often involved in signal transduction of external messages to the intracellular machinery that governs normal cell growth and differentiation. As such, the DNA sequences of proto-&#173;oncogenes are highly conserved in nature between such dif&#173;fer&#173;ent organisms as &#173;humans and yeast. Proto-&#173;oncogenes can be activated to oncogenes by loss or disruption of normal regulation.</p>
			<p class="h3 ParaOverride-16">Oncogenes</p>
			<p class="body-text--no-indent-">Oncogenes &#173;were first detected in retroviruses, which had acquired them from their host in order to take control of cell growth. &#173;These oncogenes are often identified by names that refer to the viral source, as for example, <span class="italic">ras</span> (<span class="italic">ra</span>t <span class="italic">s</span>arcoma virus). They are known to be activated not only in virus-&#173;induced malignancies but in common nonviral cancers in &#173;humans. Oncogenes behave the same way that autosomal dominant traits behave, and only 1 mutant allele is needed for tumor formation, presumably by a dominant-negative effect on regulation of signal transduction.</p>
			<p class="h3 ParaOverride-16">Tumor suppressor genes</p>
			<p class="body-text--no-indent-">Tumor suppressor genes, also called <span class="italic">antioncogenes,</span> are genes that must be pres&#173;ent in 1 functional copy to prevent uncontrolled cell proliferation. Although some may represent genes whose products participate in checkpoints for the cell cycle, a characteristic of tumor suppressor genes is the diversity of their normal functions. Examples of tumor suppressor genes include the genes for retinoblastoma, Wilms tumor, neurofibromatosis types 1 and 2, tuberous sclerosis, ataxia-&#173;telangiectasia, and von Hippel–&#173;Lindau disease. All of &#173;these examples (except ataxia-&#173;telangiectasia) behave as autosomal dominant traits, but the mechanism of tumor formation for tumor suppressor genes is very dif&#173;fer&#173;ent from that for oncogenes. If 1 allele is already defective &#173;because of a hereditary mutation, the other allele must also be lost for tumor formation to occur (also known as the <span class="italic">2-&#173;hit hy</span><span class="italic">po&#173;thesis</span>). This loss of the second allele is termed <span class="italic">loss of heterozygosity,</span> and it can occur from a second mutation, gene deletion, chromosomal loss, or mitotic recombination.</p>
			</div>
			<p class="h1 ParaOverride-5">Mitochondrial Disease</p>
			<div id="Chapt5_Top9">
			<p class="body-text--no-indent-">A significant number of disorders associated with the eye or visual system involve mitochondrial deletions and mutations. Mitochondrial diseases should be considered whenever the inheritance pattern of a trait suggests maternal transmission. Although the inheritance pattern might superficially resemble that of an X-&#173;linked trait, maternal transmission differs in that all of the offspring of affected females—&#173;both &#173;daughters and sons—&#173;can inherit the trait, but only the &#173;daughters can pass it on.</p>
			<p class="body-text">The phenotype and severity of mitochondrial disease appear to depend on the nature of the mutation, the presence or degree of heteroplasmy (coexistence of more than 1 species of mitochondrial DNA [mtDNA]—ie, wild type and mutant), and the oxidative needs of the tissues involved. Spontaneous deletions and mutations of mtDNA accumulate with age, and the effect of this accumulation is to decrease the efficiency and function of the electron transport system, reducing the availability of adenosine triphosphate (ATP). When energy production becomes insufficient to maintain the function of cells or tissue, disease occurs. &#173;There appears to be an impor&#173;tant interaction between age and tissue threshold of oxidative phosphorylation and the expression of inherited mutations of mtDNA.</p>
			<p class="body-text">With each cell division, the number of mutant mtDNA copies that are partitioned to a given &#173;daughter cell is random, unlike in mendelian inheritance. &#173;After a number of cell divisions, some cells, purely by chance, receive more normal or more mutant copies of mtDNA, resulting in a drift &#173;toward homoplasmy in subsequent cell lines. This pro&#173;cess is called <span class="italic">replicative segregation.</span> With mtDNA deletions, preferential replication of the smaller deleted molecules &#173;causes an increase of deleted copy over time. The trend &#173;toward homoplasmy helps explain why disease worsens with age and why organ systems not previously involved in multisystem mitochondrial disease become involved.</p>
			<p class="body-text">&#173;Causes of mitochondrial diseases can be categorized as follows:</p>
			<ul>
				<li class="bullet-list-first">large rearrangements of mtDNA (deletions or insertions), such as chronic progressive external ophthalmoplegia (CPEO), Kearns-&#173;Sayre syndrome, and Pearson marrow-&#173;pancreas syndrome</li>
				<li class="bullet-list-mid">mutations of mtDNA-&#173;encoded ribosomal RNA (rRNA), such as occur in maternally inherited sensorineural deafness and aminoglycoside-&#173;induced deafness</li>
				<li class="bullet-list-mid">mutations of mtDNA-&#173;encoded tRNA, such as occur in the syndromes of MELAS (<span class="bullet-list_italic">m</span>itochondrial <span class="bullet-list_italic">e</span>ncephalomyopathy, <span class="bullet-list_italic">l</span>actic <span class="bullet-list_italic">a</span>cidosis, and <span class="bullet-list_italic">s</span>trokelike episodes), MERRF (<span class="bullet-list_italic">m</span>yoclonic <span class="bullet-list_italic">e</span>pilepsy with <span class="bullet-list_italic">r</span>agged <span class="bullet-list_italic">r</span>ed <span class="bullet-list_italic">f</span>ibers), MIDD (<span class="bullet-list_italic">m</span>aternally <span class="bullet-list_italic">i</span>nherited <span class="bullet-list_italic">d</span>iabetes and <span class="bullet-list_italic">d</span>eafness), and (in about 30% of cases) CPEO</li>
				<li class="bullet-list-last ParaOverride-4">missense and nonsense mutations, such as are pres&#173;ent in Leber hereditary optic neuropathy; and neuropathy, ataxia, and RP</li>
			</ul>
			<p class="h2 ParaOverride-15">Chronic Progressive External Ophthalmoplegia</p>
			<p class="body-text--no-indent-">CPEO is a disorder involving progressive ptosis and paralysis of eye muscles associated with a ragged red myopathy, usually as a result of deletion of a portion of the mitochondrial genome. Patients with CPEO commonly have pigmentary retinopathy that does not create significant visual disability. Infrequently, they may have more marked ret&#173;i&#173;nal or other system involvement, the so-&#173;called <span class="italic">CPEO-&#173;plus syndromes.</span> In Kearns-&#173;Sayre syndrome, CPEO is associated with heart block and severe RP with marked visual impairment. Pearson marrow-&#173;pancreas syndrome results from a large deletion of mtDNA and pres&#173;ents in younger patients with an entirely dif&#173;fer&#173;ent phenotype involving sideroblastic anemia and pancreatic exocrine dysfunction. However, in patients afflicted during their &#173;later years, Pearson marrow-&#173;pancreas syndrome can pres&#173;ent with a phenotype resembling that of Kearns-&#173;Sayre syndrome.</p>
			<p class="body-text">Roughly 50% of patients with CPEO have demonstrable mtDNA deletions, whereas virtually all patients with Kearns-&#173;Sayre syndrome have large deletions. Of patients with CPEO who do not harbor demonstrable mtDNA deletions, up to 30% may have a point mutation at nucleotide position 3243, the same mutation in the tRNA for leucine that in other individuals is associated with MELAS syndrome. For all syndromes associated with deletions, such as Kearns-&#173;Sayre and CPEO, detection of the deletion usually requires study of the muscle tissue.</p>
			<p class="h2 ParaOverride-15">MELAS and MIDD</p>
			<p class="body-text--no-indent-">Two dif&#173;fer&#173;ent disorders—&#173;mitochondrial encephalomyopathy, lactic acidosis, and strokelike episodes (MELAS) and maternally inherited diabetes and deafness (MIDD; also called <span class="italic">type 2 diabetes mellitus with deafness</span>)—&#173;are associated with an mtDNA point mutation (A-&#173;to-&#173;G change at nucleotide position 3243), which affects an mtDNA-&#173;encoded tRNA. Macular ret&#173;i&#173;nal pigment epithelial atrophy and this mutation have been described in patients with MELAS.</p>
			<p class="reference--journal--first">Isashiki Y, Nakagawa M, Ohba N, et&#160;al. Ret&#173;i&#173;nal manifestations in mitochondrial diseases associated with mitochondrial DNA mutation. <span class="reference--journal-_italic">Acta Ophthalmol Scand.</span> 1998;76(1):6–13.</p>
			<p class="reference--journal--last ParaOverride-9">Yu-&#173;Wai-&#173;Man P, Griffiths PG, Hudson G, Chinnery PF. Inherited mitochondrial optic neuropathies. <span class="reference--journal-_italic">J Med Genet.</span> 2009;46(3):145–158.</p>
			<p class="h2 ParaOverride-15">Leber Hereditary Optic Neuropathy</p>
			<p class="body-text--no-indent-">The most impor&#173;tant ophthalmic disease of mitochondria is Leber hereditary optic neuropathy (LHON), which is more prevalent in males than in females but does not fit a classic X-&#173;linked pattern of transmission. The trait is not transmitted to the offspring of affected males, but virtually &#173;every &#173;daughter and son of a female patient with LHON inherits the trait. In approximately 50% of cases, LHON development is correlated with a single base change (G to A at nucleotide position 11778&#160;in the <span class="italic">ND-4</span> gene) in &#173;human mtDNA involved in the synthesis of NADH dehydrogenase. In addition to optic atrophy, patients can exhibit peripapillary microangiopathy. LHON can also occur from other so-&#173;called primary mutations at nucleotide positions 3460 of <span class="italic">ND-1</span> and 14484 of <span class="italic">ND-6,</span> as well as several other rare mutations. At least 12 secondary mutations have been associated with LHON, often when multiple mutations are pres&#173;ent in an individual’s mitochondria. Some authors think that &#173;these secondary mutations cause disease by additive detrimental effects on the electron transport system of oxidative phosphorylation. Most of &#173;these secondary mutations appear in the general population, and debate persists on &#173;whether each mutation alone is truly pathogenic.</p>
			<p class="body-text">The likelihood of improvement of visual acuity over time appears to differ among patients with the separate mutations associated with LHON. Mutation at nucleotide position 11778 is associated with the least likelihood of recovery, and mutation at nucleotide position 14484 is associated with the greatest likelihood.</p>
			<p class="h2 ParaOverride-8">Neuropathy, Ataxia, and Retinitis Pigmentosa</p>
			<p class="body-text--no-indent-">Neuropathy, ataxia, and retinitis pigmentosa (NARP) is associated with a single base-&#173;pair mutation at nucleotide position 8993&#160;in the <span class="italic">ATPase-6</span> gene. The NARP phenotype occurs when the percentage of mutant mtDNA is less than 80%, whereas the same mutation pres&#173;ent at much higher proportions (greater than 95%) can cause Leigh syndrome, a severe neurodegenerative disease of infancy and early childhood. The 8993 mutation is demonstrable in fibroblasts and lymphoblasts.</p>
			</div>
			<p class="h1 ParaOverride-5">The Search for Genes in Specific Diseases</p>
			<div id="Chapt5_Top10">
			<p class="body-text--no-indent-">A variety of methods have been used to assign individual genes to specific chromosomes, to link individual genes to one another, and to link diseases to specific genes.</p>
			<p class="h2 ParaOverride-8">Polymerase Chain Reaction</p>
			<p class="body-text--no-indent-">Polymerase chain reaction (PCR) is a technique used in molecular biology to amplify a single copy or a few copies of a segment of DNA or RNA by several &#173;orders of magnitude, generating thousands to millions of copies of a par&#173;tic&#173;u&#173;lar DNA sequence. PCR is a common and indispensable technique used in clinical and research laboratories for a broad variety of applications. Clinically, PCR has been utilized in establishing the etiology of ocular infections. For example, PCR performed on ocular fluids can detect numerous members of the herpes virus &#173;family.</p>
			<p class="body-text">PCR methods rely on thermal cycling, which involves exposing the reactants to repeated cycles of heating and cooling, permitting dif&#173;fer&#173;ent temperature-&#173;dependent reactions of DNA melting and enzyme-&#173;driven DNA replication. Primers (short DNA fragments) containing sequences complementary to the target region, along with a DNA polymerase (usually Taq polymerase), enable selective and repeated amplification. As PCR progresses, the DNA generated is itself used as a template for replication, setting in motion a chain reaction in which the original DNA template is exponentially amplified.</p>
			<p class="reference--journal--single ParaOverride-9">Sugita S, Ogawa M, Shimizu N, et&#160;al. Use of a comprehensive polymerase chain reaction system for diagnosis of ocular infectious diseases. <span class="reference--journal-_italic">Ophthalmology.</span> 2013;120(9):1761–1768.</p>
			<p class="h2 ParaOverride-15">Ge&#173;ne&#173;tic Markers</p>
			<p class="body-text--no-indent-">Occasionally in cytoge&#173;ne&#173;tic studies, a ge&#173;ne&#173;tic marker such as a large deletion or translocation (eg, 11p13&#160;in aniridia) may be vis&#173;i&#173;ble. Other markers used to identify the location of genes include blood groups (eg, as in Duffy blood group and Coppock cataract); restriction fragment length polymorphisms (RFLPs; eg, as in RP); microsatellites of variable number of tandem repeats (VNTRs); and most recently SNPs, as used in many GWAS. Cytoge&#173;ne&#173;tic tests are conducted on white blood cells, whereas the other ge&#173;ne&#173;tic markers test DNA that is extracted most commonly from peripheral blood or saliva.</p>
			<p class="body-text">If a specific chromosomal structure is abnormal or even normally variant, its transmission through a &#173;family with a hereditary disease, as mapped by a pedigree, may support the assumption that the mutant gene and the variant chromosome are comigrating. Thus, the mutant gene is likely to be physically located on the variant chromosome—&#173;that is, the gene is a cytoge&#173;ne&#173;tic marker for the disease.</p>
			<p class="h2 ParaOverride-15">Gene Dosage</p>
			<p class="body-text--no-indent-">If a portion of a chromosome containing a specific gene is deleted, the amount of the gene product &#173;will be determined only by the remaining homologue. For example, &#173;people with an interstitial deletion of part of the long arm of chromosome 13 may have serum levels of esterase D that are 50% of normal. When several such individuals &#173;were also found to have retinoblastoma, it was suggested that both the esterase and the retinoblastoma genes are located in the missing segment. In contrast to the reduced activity caused by a deletion, a duplication may produce 150% of normal activity of a given gene product, as a result of &#173;either a chromosomal trisomy or a triplication of a specific chromosomal segment. Gene dosage appears to be a mechanism of disease in anterior segment dysgenesis, caused by duplication or deletion of the <span class="italic">FOXC1</span> gene; both 50% and 150% of the transcription &#173;factor lead to this dysgenesis.</p>
			<p class="h2 ParaOverride-15">Linkage and Disease Association</p>
			<p class="body-text--no-indent-">Even if no information is known about the nature or function of a gene for a disease, linkage studies may be able to localize the gene to a given chromosome or a specific marker. In 1937, Bell and Haldane recognized the first linkage between 2 diseases on a &#173;human chromosome: congenital color vision deficiency and hemophilia on the X chromosome. Subsequent investigations have led to the chromosomal mapping of a large number of dif&#173;fer&#173;ent &#173;human ocular diseases.</p>
			<p class="h3 ParaOverride-16">Gene assignments</p>
			<p class="body-text--no-indent-">&#173;Every chromosome has numerous defined genes. The &#173;Human Genome Proj&#173;ect identified and mapped approximately 20,000–25,000 genes. In addition, the database Online Mendelian Inheritance in Man, OMIM (<a href="https://omim.org"><span class="Hyperlink">https://omim.org</span></a>), lists information on all known mendelian disorders. &#173;Human gene mapping has 2 major applications. The first is identification of the gene for a specific ge&#173;ne&#173;tic disease by its linkage to a known marker. For example, suppose gene A &#173;causes a hereditary disease and gene B is a known enzyme or polymorphic marker closely linked to A. Even though no biochemical test exists for A, a tight linkage to B would allow a reasonable probability of identifying the disease for prenatal diagnosis and sometimes for carrier detection. The second impact of mapping is as an aid to understanding the cause of the phenotypic malformations in specific chromosomal diseases. For example, the phenotype of Down syndrome may result from triplication of only the distal long arm of chromosome 21 through a chromosome rearrangement rather than trisomy of the entire chromosome.</p>
			<p class="body-text">It is pos&#173;si&#173;ble to detect linkage by observing the frequency with which a polymorphic marker is inherited with a disease trait. The physical distance represented by 1 cM corresponds to approximately 1 million bp (1000 kb) and to a 1% chance that recombination &#173;will result from a single meiosis (a 0.01 recombination fraction). When a ge&#173;ne&#173;tic marker is sufficiently close to a disease gene, both are rarely separated by meiotic recombination. The frequency of this separation by chromosomal exchange at meiosis is termed the <span class="italic">recombination frequency.</span> To be linked, markers should be no more than about 20 cM apart. For perspective, the average chromosome contains about 150 cM, and &#173;there are approximately 3300 cM in the entire &#173;human genome, which corresponds to 3 <span class="symbol">×</span> 10<span class="superscript _idGenCharOverride-1">9</span> bp.</p>
			<p class="body-text">When determining linkage between a gene and a marker, ge&#173;ne&#173;ticists compare dif&#173;fer&#173;ent models by calculating likelihood ratios. When the likelihood ratio is 1000:1 that the odds of one model are greater than &#173;those of another, the first is accepted over the second. The base 10 logarithm of the likelihood ratio (<span class="italic">LOD score; l</span>ogarithm of <span class="italic">od</span>ds score) is usually reported. An LOD score of 1–2 is of potential interest in terms of linkage; 2–3 is suggestive; and greater than 3 is generally considered proof of linkage. Although an LOD score of 3 gives a probability ratio of 1000:1&#160;in &#173;favor of linkage versus in&#173;de&#173;pen&#173;dent assortment, this score does not indicate a type I error as low as 0.001 but, in fact, indicates an error that is close to 0.05, the standard significance level used in statistics. (BCSC Section&#160;1, <span class="italic">Update on General Medicine,</span> explains &#173;these concepts in depth.)</p>
			<p class="h2">Candidate Gene Approaches</p>
			<p class="h3-h2">Candidate gene screening</p>
			<p class="body-text--no-indent-">The pro&#173;cess of candidate gene screening involves screening for mutations of genes that are abundantly expressed within a tissue and are &#173;either impor&#173;tant for function or specifically expressed only in that tissue. Sometimes, the candidate gene is one that recapitulates the &#173;human disease in transgenic animals. Examples of candidate gene screening discoveries include the findings of mutations of peripherin/<span class="italic">RDS</span> in autosomal dominant RP and macular dystrophies and the finding of mutations of the rod cyclic guanosine monophosphate (cGMP) <span class="greek">β</span>-&#173;subunit of rod phosphodiesterase and the cGMP–&#173;gated cation channel in autosomal recessive RP.</p>
			<p class="h3">Positional candidate gene screening</p>
			<p class="body-text--no-indent-">Whenever linkage studies localize a gene to a given chromosomal region, genes already known to reside in the same region become candidate genes for that disease. Following are some examples of disease localization that resulted from linkage to a given region, which in turn led to finding the disease-&#173;causing gene by screening for mutations of genes in the region: autosomal dominant RP from rhodopsin mutations (3q); Sorsby fundus dystrophy from <span class="italic">TIMP3</span> mutations (22q); and Oguchi disease from point deletions within the arrestin gene (2q).</p>
			</div>
			<p class="h1">Mutation Screening</p>
			<div id="Chapt5_Top11">
			<p class="h2-h1">Direct Sequencing</p>
			<p class="body-text--no-indent-">The development of techniques for rapid sequencing of DNA was one of the most significant advances in molecular ge&#173;ne&#173;tics. Currently, it costs far less to sequence a stretch of DNA than to sequence and characterize the amino acid peptide that the DNA produces.</p>
			<p class="body-text">Although other mutation screening techniques exist, sequencing of DNA is the surest and most direct. Sequencing of complementary DNA (cDNA) derived from mRNA provides a quick look at the reading frames (exons) of the gene, whereas sequencing of genomic DNA is more time-&#173;consuming &#173;because of the presence of introns between the exons. The intron–&#173;exon bound&#173;aries must be known and multiple PCR assays set up in order to screen not only the exons and their splice junction sites but also upstream and downstream regions that may be impor&#173;tant for gene activation and regulation.</p>
			<p class="body-text">DNA sequencing techniques currently in use include the enzymatic (or Sanger sequencing) method, which can be automated (<span class="xref-figure">Figure&#160;5-4</span> illustrates this procedure), and next-&#173;generation sequencing (NGS), also known as <span class="italic">massively parallel sequencing.</span> NGS offers the ability to sequence the entire genome of an individual. Some NGS methods use as probes allele-&#173;specific oligonucleotides that are constructed to employ hybridization to recognize a specific DNA sequence in order to detect a specific point mutation (<span class="xref-figure">Fig 5-5</span>).</p>
			<p class="body-text">Early methods of mutation detection included Sanger sequencing using radioactive and &#173;later fluo&#173;rescent probes; the single-&#173;stranded conformational polymorphism (SSCP) technique; denaturing gradient gel electrophoresis (DGGE); and the use of RFLPs.</p>
			<p class="body-text">Whole-&#173;exome sequencing &#173;will identify many potential mutations; however, considerable bioinformatic information is required to identify true disease-&#173;causing mutations.</p>
			<p class="reference--journal--single ParaOverride-9">Zhang J, Chiodini R, Badr A, Zhang&#160;G. The impact of next-&#173;generation sequencing on geno&#173;mics. <span class="reference--journal-_italic">J Genet Genomics.</span> 2011;38(3):95–109.</p>
			<p class="h2">Genome-&#173;Wide Association Studies</p>
			<p class="body-text--no-indent-">Although karyotyping and linkage analy&#173;sis can still be used to identify disease-&#173;associated genes, most research is now centered on GWAS and NGS. The <span class="italic">International HapMap Proj&#173;ect,</span> which followed the creation of the &#173;human gene map, compared the DNA sequence of 1184 reference individuals from 11 global populations (creating a cata&#173;log called the <span class="italic">HapMap</span>) to identify regions of variation between individuals and racial groups. By using the HapMap to study individuals from a similar population, ge&#173;ne&#173;tics researchers find that many &#173;people &#173;will share a series of SNPs or a haplotype. Thus, it is pos&#173;si&#173;ble to test only one or a few SNPs but to infer a large number of adjacent SNPs by imputation. Chip or bead platforms enable the investigation of 100,000 to millions of SNPs across the genome, forming the basis of a GWAS.</p>
			<p class="body-text">Results of a GWAS are usually presented in a <span class="italic">Manhattan plot,</span> so named &#173;because it brings to mind the New York City skyline. In a Manhattan plot, the chromosomes are arranged in order along the x-&#173;axis, and the <span class="italic">P</span> value (as <span class="symbol">−</span>log P) of the association of the disease or trait with the par&#173;tic&#173;u&#173;lar SNP at that chromosomal location is given on the y-&#173;axis. <span class="xref-figure">Figure&#160;5-6</span> shows a Manhattan plot for glaucoma. A significant gene association (threshold ≈5 <span class="symbol">×</span> 10<span class="superscript _idGenCharOverride-1">–8</span>) &#173;will often have multiple adjacent SNPs at high levels of significance, and thus a column of points &#173;will rise on the plot. It is rare that the SNPs themselves are the disease-&#173;causing mutations. Usually they are linked in the haplotype to the mutation, which is why researchers &#173;will then use fine-&#173;mapping of the region by looking at a large number of SNPs in the nearby region.</p>
			<p class="body-text">Combining numerous studies, usually of multiple ethnic groups, in meta-&#173;analyses allows for identification of additional associated gene regions. <span class="xref-figure">Figure&#160;5-7</span> shows how GWAS meta-&#173;analyses combine data from individual GWAS. <span class="xref-figure">Figure&#160;5-8</span> shows the meta-&#173;analysis of GWAS for age-&#173;related macular degeneration (AMD), with 19 loci now identified. The effect size for all of &#173;these genes is usually small, but cumulatively they account for approximately 40% of AMD heritability.</p>
			<p class="body-text">Comparison of GWAS from dif&#173;fer&#173;ent ethnic groups can help clarify &#173;whether the SNP itself is disease causing or just linked to the true disease-&#173;causing mutation(s). An example is the <span class="italic">LOXL1</span> gene, which is associated with pseudoexfoliation. One SNP was associated with disease in the Caucasian population, but disease was associated with the alternate SNP in the Japa&#173;nese population (<span class="xref-figure">Fig 5-9A</span>). Thus, it is unlikely that this SNP is actually disease causing but more likely that dif&#173;fer&#173;ent SNPs are associated with the true disease-&#173;causing mutation in East Asian and white, or Caucasian-&#173;derived, populations. In contrast, another SNP had equivalent association in both populations (<span class="xref-figure-local">Fig 5-9B</span>).</p>
			<p class="body-text">For a cata&#173;log of GWAS including ophthalmic studies, see the Eu&#173;ro&#173;pean Molecular Biology Laboratory–&#173;European Bioinformatics Institute (EMBL-&#173;EBI) cata&#173;log at <a href="http://www.ebi.ac.uk/gwas/"><span class="Hyperlink">http://&#173;www.&#173;ebi.&#173;ac.&#173;uk/&#173;gwas/&#173;</span></a>.</p>
			<p class="h2 ParaOverride-17">Determining &#173;Whether Ge&#173;ne&#173;tic Change Is a Pathogenic Mutation</p>
			<p class="body-text--no-indent-">If a patient is to be considered for a gene-&#173;based therapy, it is impor&#173;tant for the clinician to understand how bioinformatics weights the likelihood that a ge&#173;ne&#173;tic change is a pathogenic mutation. With the huge amount of ge&#173;ne&#173;tic information provided by whole-&#173;exome sequencing, whole-&#173;genome sequencing, and genome arrays used in GWAS, many variants of unknown significance have been identified. Numerous types of evidence are used to distinguish a benign polymorphism from a pathogenic mutation. &#173;These include population data on the frequency of a variant in cases and controls; segregation data in pedigrees; computational and predictive data, which include SIFT (sorting intolerant from tolerant) and PolyPhen (polymorphism phenotyping); and functional data from cell and animal models (&#173;Table&#160;5-1).</p>
			<p class="reference--journal--single">Stone EM, Andorf JL, Whitmore SS, et&#160;al. Clinically focused molecular investigation of 1000 consecutive families with inherited ret&#173;i&#173;nal disease. <span class="italic">Ophthalmology</span>. 2017;124(9):1314–1331.</p>
			</div>
			<p class="h1 ParaOverride-18">Gene Therapy</p>
			<div id="Chapt5_Top12">
			<p class="body-text--no-indent-">Gene therapy holds much promise, but the field remains in its infancy. The potential for cure is not matched by &#173;either technology or understanding. Key challenges remain in characterizing mutations of genes for major diseases, understanding the pathogenic relevance of identified genes, and developing proper delivery systems for curative gene constructs (the main long-&#173;term gene therapy vehicle—&#173;viruses—is currently limited by the size of the gene, inflammatory effects, and the risk of oncogenesis).</p>
			<p class="h2 ParaOverride-17">Replacement of Absent Gene Product in X-&#173;Linked and Recessive Diseases</p>
			<p class="body-text--no-indent-">For ge&#173;ne&#173;tic diseases in which the mutant allele produces &#173;either no message or an in&#173;effec&#173;tive gene product (called a <span class="italic">null allele</span>), correction of the disorder may be pos&#173;si&#173;ble by &#173;simple replacement of the gene in the deficient cells or tissues. It is theoretically pos&#173;si&#173;ble to transfer normal genes into &#173;human cells that harbor &#173;either null or mutant genes not producing a stable, translated product. Vectors used to carry the ge&#173;ne&#173;tic material into the cells include adenoviruses, retroviruses (especially adeno-&#173;associated viruses [AAVs]), and plasmid–&#173;liposome complexes. AAV vector gene therapy has been successful in curing many disorders in animal models, such as the <span class="italic">RPE65</span> gene mutation that &#173;causes RP in the Briard dog.</p>
			<p class="body-text">&#173;Human gene therapy &#173;trials with <span class="italic">RPE65</span> suggest no major early adverse effects and some improvement in visual function. In 2017, the US Food and Drug Administration approved the use of voretigene neparvovec (AAV2-&#173;hRPE65v2) in patients with confirmed biallelic <span class="italic">RPE65</span>-&#173;mediated ret&#173;i&#173;nal dystrophy (ie, Leber congenital amaurosis [LCA] and RP). The current cost of treating both eyes is $850,000. Also, mutations in <span class="italic">RPE65</span> account for only a small percentage of LCA (see Chapter 6, Fig 6-4); thus, this therapy is not suitable for all patients with LCA. Studies in younger subjects (<span class="symbol">&lt;</span>3&#160;years) are also &#173;under way, and several other ret&#173;i&#173;nal dystrophy genes are &#173;under investigation for &#173;human gene therapy &#173;trials.</p>
			<p class="reference--journal--first">Carvalho LS, Vandenberghe LH. Promising and delivering gene therapies for vision loss. <span class="reference--journal-_italic">Vision Res.</span> 2015;111(Pt B):124–133.</p>
			<p class="reference--journal--last ParaOverride-9">Russell S, Bennett J, Wellman JA, et&#160;al. Efficacy and safety of voretigene neparvovec (AAV2-&#173;hRPE65v2) in patients with RPE65-&#173;mediated inherited ret&#173;i&#173;nal dystrophy: a randomised, controlled, open-&#173;label, phase 3 trial. <span class="reference--journal-_italic">Lancet</span>. 2017;390(10097):849–860.</p>
			<p class="h2">Strategies for Dominant Diseases</p>
			<p class="body-text--no-indent-">Dominant diseases are caused by production of a gene product that is &#173;either insufficient <span class="italic">(haploid insufficiency)</span> or conducive to disease <span class="italic">(dominant-&#173;negative effect).</span> Theoretically, haploid insufficiency should be treatable by gene replacement therapy as outlined in the previous section for X-&#173;linked and recessive diseases. For dominant disorders produced by defective developmental genes, this correction would have to occur in early fetal development.</p>
			<p class="body-text">Disorders resulting from a dominant-&#173;negative effect require a dif&#173;fer&#173;ent approach. Thus, strategies for treatment of dominant disease differ, depending on &#173;whether a functional gene product is produced. Some genes code for RNA molecules that can bind to mRNA from another gene and block the other molecule’s ability to be translated. Greater understanding of &#173;these genes may enable the creation of &#173;either drugs or new gene-&#173;encoded RNA molecules that can block the translation of mRNA for defective alleles, thus allowing only the normal allele to be expressed.</p>
			<p class="body-text">Another approach is the use of oligonucleotides that are designed to bind with mRNA from mutant alleles, stopping the mRNA from being translated by ribosomes (<span class="xref-figure">Fig 5-10</span>). Although many prob&#173;lems need to be resolved for such therapy to be effective, this approach holds promise for autosomal dominant disorders in which disease is caused by expression of the mutant gene product.</p>
			<p class="body-text">One approach to the treatment of autosomal dominant diseases is to target the translated strand of the mutant allele by antisense DNA, a sequence of DNA designed to &#173;anneal the abnormal mRNA and block its processing or translation. The use of ribozymes, RNA molecules that have the ability to cleave certain RNA molecules, provides another approach. A third method utilizes <span class="italic">short interfering RNA (siRNA),</span> also known as <span class="italic">small interference RNA,</span> to bind to mRNA and lead to the eventual degradation of specific mRNA molecules. The use of siRNA molecules as potential therapeutic agents has become increasingly popu&#173;lar, and this approach has proven to be a power&#173;ful means by which to study the function of novel gene products. However, one challenge with siRNA therapy is achieving intracellular delivery. Another challenge is cell-&#173;surface TLR3 receptor stimulation, which can induce immune or antiangiogenic pro&#173;cesses as a generic class property.</p>
			<p class="body-text">A new form of genome editing known as <span class="italic">CRISPR–&#173;Cas9</span> (clustered, regularly interspaced, short palindromic repeats–&#173;CRISPR-&#173;associated protein 9) has been used to correct point mutations in the DNA sequence of cells. Combining the technology of CRISPR<span class="italic">–&#173;</span>Cas9 with that of induced pluripotent stem cells (iPSCs) could potentially allow a scenario in which a skin biopsy is performed on a patient with an inherited ret&#173;i&#173;nal disease, skin cells are induced to produce pluripotent stem cells, and the causative mutation is edited out with CRISPR–&#173;Cas9. The cells could then be grown into the appropriate ret&#173;i&#173;nal cell line and implanted in the diseased eye. Before clinical &#173;trials commence, this personalized therapy, which would be costly, must still overcome issues with immunity and the risk of tumor development.</p>
			<p class="reference--journal--first">Burnight ER, Gupta M, Wiley LA, et&#160;al. Using CRISPR-&#173;Cas9 to generate gene-&#173;corrected autologous iPSCs for the treatment of inherited ret&#173;i&#173;nal degeneration. <span class="italic">Mol Ther</span>. 2017;<br />25(9):1999–2013.</p>
			<p class="reference--journal--last">Hung SSC, McCaughey T, Swann O, P<span class="accent">é</span>bay A, Hewitt AW. Genome engineering in ophthalmology: application of CRISPR/Cas to the treatment of eye disease. <span class="italic">Prog Retin Eye Res</span>. 2016;53:1–20.</p>
		</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer005">
				<div id="_idContainer003" class="Basic-Text-Frame">
					<p class="figure-caption"><span class="figure-number">Figure&#160;5-1</span> In the cell cycle, the progression from DNA synthesis (S) to mitosis (M) includes phases before (G<span class="figure-caption_subscript _idGenCharOverride-1">1</span>) and &#173;after (G<span class="figure-caption_subscript _idGenCharOverride-1">2</span>) the replication of DNA. Upon receiving signals to differentiate, cells leave the cycle and enter the final stage of cell differentiation, or terminal differentiation. Regulation of the cycle occurs at numerous checkpoints, before the cell progresses from one phase to another. &#173;Under certain circumstances, cells may return to quiescence (G<span class="figure-caption_subscript _idGenCharOverride-1">0</span>) or enter the pathway to programmed cell death (apoptosis).</p>
				</div>
				<div id="_idContainer004" class="_idGenObjectStyleOverride-1">
					<img class="_idGenObjectAttribute-1" src="BCSC2019_S02_C05_p169_198_9P-web-resources/image/fig_5-1.png" alt="" />
				</div>
			</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer008">
				<div id="_idContainer006" class="Basic-Text-Frame">
					<p class="figure-caption"><span class="figure-number">Figure&#160;5-2</span> Structures of the cell showing the location of DNA within chromosomes and mitochondria. The basic double helix of nucleotides is divided into noncoding regions, including introns and promoter regions, and coding exons, which form genes. The figure shows a noncoding intron between 2 exons. The intron is spliced out before the segment is translated. This modification occurs following transcription, though before messenger RNA (mRNA) is finalized.</p>
				</div>
				<div id="_idContainer007" class="_idGenObjectStyleOverride-1">
					<img class="_idGenObjectAttribute-1" src="BCSC2019_S02_C05_p169_198_9P-web-resources/image/fig_5-2.png" alt="" />
				</div>
			</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer011">
				<div id="_idContainer009" class="Basic-Text-Frame">
					<p class="figure-caption"><span class="figure-number">Figure&#160;5-3</span> The central dogma of ge&#173;ne&#173;tics, as represented schematically, is that DNA sequence codes are transcribed to the mRNA sequence, and then the mRNA transcription is translated into the amino acid sequence of the coded protein. However, proteins in the form of transcription &#173;factors and complementary short RNA sequences can modify translation and transcription. &#173;These proteins are being investigated as potential forms of therapy.</p>
				</div>
				<div id="_idContainer010" class="_idGenObjectStyleOverride-1">
					<img class="_idGenObjectAttribute-1" src="BCSC2019_S02_C05_p169_198_9P-web-resources/image/fig_5-3.png" alt="" />
				</div>
			</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer014">
				<div id="_idContainer012" class="Basic-Text-Frame">
					<p class="figure-caption"><span class="figure-number">Figure&#160;5-4</span> Schematic repre&#173;sen&#173;ta&#173;tion of the original Sanger dideoxy chain-&#173;termination sequencing method. The results produced are shown in step 3: DNA fragments resolved on a polyacrylamide gel and a sequencing trace from a modern automated sequencing machine. <span class="figure-source-note">(Original figure from Oxbridge Biotech Roundtable; redrawn by Mark Miller.)</span></p>
				</div>
				<div id="_idContainer013" class="_idGenObjectStyleOverride-1">
					<img class="_idGenObjectAttribute-1" src="BCSC2019_S02_C05_p169_198_9P-web-resources/image/fig_5-4.png" alt="" />
				</div>
			</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer017">
				<div id="_idContainer015" class="Basic-Text-Frame">
					<p class="figure-caption"><span class="figure-number">Figure&#160;5-5</span> Schematic summary of the whole-&#173;genome shotgun sequencing method of next-&#173;generation sequencing (NGS). At a basic level, all NGS technologies use the same princi&#173;ple: fragment the DNA, add primers/adapters, amplify, and sequence. In whole-&#173;genome shotgun sequencing, a DNA sample is randomly broken into numerous small fragments that are then sequenced using the chain-&#173;termination method. Multiple overlapping DNA fragments produced from numerous repetitions of this pro&#173;cess are then assembled into a single continuous sequence on a computer program. <span class="figure-source-note">(Original figure from Oxbridge Biotech Roundtable; redrawn by Mark Miller.)</span></p>
				</div>
				<div id="_idContainer016" class="_idGenObjectStyleOverride-1">
					<img class="_idGenObjectAttribute-1" src="BCSC2019_S02_C05_p169_198_9P-web-resources/image/fig_5-5.png" alt="" />
				</div>
			</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer020">
				<div id="_idContainer018" class="_idGenObjectStyleOverride-1">
					<img class="_idGenObjectAttribute-1" src="BCSC2019_S02_C05_p169_198_9P-web-resources/image/fig_5-6.png" alt="" />
				</div>
				<div id="_idContainer019" class="Basic-Text-Frame">
					<p class="figure-caption"><span class="figure-number">Figure&#160;5-6</span> Manhattan plot for glaucoma, identifying the <span class="figure-caption_italic">CDKN2BAS</span> region at 9p21 and the <span class="figure-caption_italic">SIX1/SIX6</span> region at 14q23. <span class="figure-source-note">(Reproduced with permission from Wiggs JL, Yaspan BL, Hauser MA, et&#160;al. Common variants at 9p21 and 8q22 are associated with increased susceptibility to optic nerve degeneration in glaucoma.</span> <span class="figure-source-emphasis">PLoS Genet.</span> <span class="figure-source-note">2012;8(4):e1002654.)</span></p>
				</div>
			</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer023">
				<div id="_idContainer021" class="Basic-Text-Frame">
					<p class="figure-caption"><span class="figure-number">Figure&#160;5-7</span> Meta-&#173;analysis of Manhattan plots. <span class="figure-source-note">(Reproduced with permission from Manolio TA. Genomewide association studies and assessment of the risk of disease.</span> <span class="figure-source-emphasis">N Engl J Med.</span> <span class="figure-source-note">2010;363(2):166–176.)</span></p>
				</div>
				<div id="_idContainer022" class="_idGenObjectStyleOverride-1">
					<img class="_idGenObjectAttribute-1" src="BCSC2019_S02_C05_p169_198_9P-web-resources/image/fig_5-7.png" alt="" />
				</div>
			</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer026">
				<div id="_idContainer024" class="Basic-Text-Frame">
					<p class="figure-caption"><span class="figure-number">Figure&#160;5-8</span> Manhattan plot for age-&#173;related macular degeneration meta-&#173;analysis identifying numerous associated genes. <span class="figure-source-note">(Reproduced with permission from Fritsche LG, Chen W, Schu M, et&#160;al. Seven new loci associated with age-&#173;related macular degeneration.</span> <span class="figure-source-emphasis">Nat Genet.</span> <span class="figure-source-note">2013;45(4):433–439.)</span></p>
				</div>
				<div id="_idContainer025" class="_idGenObjectStyleOverride-1">
					<img class="_idGenObjectAttribute-1" src="BCSC2019_S02_C05_p169_198_9P-web-resources/image/fig_5-8.png" alt="" />
				</div>
			</div>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer029">
				<div id="_idContainer027" class="Basic-Text-Frame">
					<p class="figure-caption"><span class="figure-number">Figure&#160;5-9</span> Forest plots <span class="figure-caption_italic">(at right)</span> of meta-&#173;analyses for single nucleotide polymorphisms (SNPs) near <span class="figure-caption_italic">LOX1</span> in pseudoexfoliation syndrome (XFS) and pseudoexfoliation glaucoma (XFG). A forest plot is a graphical display designed to illustrate the relative strength of effects found in dif&#173;fer&#173;ent quantitative scientific studies that address the same question; essentially, it graphically represents a meta-&#173;analysis of the results. <span class="figure-caption_bold">A,</span> Meta-&#173;analysis of the association of SNP rs1048661 with a combined group of XFS and XFG cases. Subgroup meta-&#173;analysis indicated that the odds ratios (ORs) of SNP rs1048661 G allele are reversed in Caucasian and Japa&#173;nese populations. <span class="figure-caption_bold">B,</span> Meta-&#173;analysis of the association of SNP rs3825942 with a combined group of XFS and XFG cases. Subgroup meta-&#173;analysis indicated that the ORs of SNP rs3825942 G allele are consistent in Caucasian and Japa&#173;nese populations. Square <span class="figure-caption_symbol">=</span> study-&#173;specific OR, with the size of the square proportional to the weight of the study; horizontal line <span class="figure-caption_symbol">=</span> 95% confidence interval (CI); diamond <span class="figure-caption_symbol">=</span> summary OR with its corresponding 95% CI. <span class="figure-source-note">(Reproduced with permission from Chen H, Chen LJ, Zhang M, et&#160;al. Ethnicity-&#173;based subgroup meta-&#173;analysis of the association of</span> <span class="figure-source-emphasis">LOXL1</span> <span class="figure-source-note">polymorphisms with glaucoma.</span> <span class="figure-source-emphasis">Mol Vis.</span> <span class="figure-source-note">2010;16:167–177.)</span></p>
				</div>
				<div id="_idContainer028" class="_idGenObjectStyleOverride-1">
					<img class="_idGenObjectAttribute-1" src="BCSC2019_S02_C05_p169_198_9P-web-resources/image/fig_5-9.png" alt="" />
				</div>
			</div>
		</div>
		<div id="_idContainer030" class="Basic-Text-Frame">
			<table id="table001" class="Basic-Table">
				<colgroup>
					<col class="_idGenTableRowColumn-1" />
					<col class="_idGenTableRowColumn-2" />
					<col class="_idGenTableRowColumn-3" />
					<col class="_idGenTableRowColumn-2" />
					<col class="_idGenTableRowColumn-3" />
					<col class="_idGenTableRowColumn-2" />
					<col class="_idGenTableRowColumn-3" />
				</colgroup>
				<tbody>
					<tr class="Basic-Table _idGenTableRowColumn-4">
						<td class="Basic-Table CellOverride-1" colspan="7">
							<p class="table-title"><span class="table-number">&#173;Table&#160;5-1</span> Does a Change in a Gene &#173;Really Cause Disease?</p>
						</td>
					</tr>
					<tr class="Basic-Table _idGenTableRowColumn-5">
						<td class="Basic-Table CellOverride-2" />
						<td class="Basic-Table CellOverride-2" />
						<td class="Basic-Table CellOverride-3">
							<p class="table-column-head">Strong Evidence That a DNA Change Is Benign</p>
						</td>
						<td class="Basic-Table CellOverride-2" />
						<td class="Basic-Table CellOverride-2">
							<p class="table-column-head">Moderate Evidence That a DNA Change Is Pathogenic</p>
						</td>
						<td class="Basic-Table CellOverride-2" />
						<td class="Basic-Table CellOverride-2">
							<p class="table-column-head">Strong Evidence That a DNA Change Is Pathogenic</p>
						</td>
					</tr>
					<tr class="Basic-Table _idGenTableRowColumn-6">
						<td class="Basic-Table CellOverride-4">
							<p class="table-body"><span class="table-body_bold">Population data</span></p>
						</td>
						<td class="Basic-Table CellOverride-4" />
						<td class="Basic-Table CellOverride-4">
							<p class="table-body">eg, Frequency of the DNA change is too high in controls</p>
						</td>
						<td class="Basic-Table CellOverride-4" />
						<td class="Basic-Table CellOverride-4">
							<p class="table-body">eg, DNA change is absent in population databases</p>
						</td>
						<td class="Basic-Table CellOverride-4" />
						<td class="Basic-Table CellOverride-4">
							<p class="table-body">eg, Frequency of the DNA change in affected cases is statistically increased over that in controls</p>
						</td>
					</tr>
					<tr class="Basic-Table _idGenTableRowColumn-7">
						<td class="Basic-Table CellOverride-5">
							<p class="table-body"><span class="table-body_bold">Segregation data</span></p>
						</td>
						<td class="Basic-Table CellOverride-5" />
						<td class="Basic-Table CellOverride-5">
							<p class="table-body">eg, No segregation of DNA change with disease in families</p>
						</td>
						<td class="Basic-Table CellOverride-5" />
						<td class="Basic-Table CellOverride-5">
							<p class="table-body">eg, Odds of DNA change and disease occurring together are <span class="table-body_symbol">&gt;</span>1&#160;in 16&#160;in a &#173;family</p>
						</td>
						<td class="Basic-Table CellOverride-5" />
						<td class="Basic-Table CellOverride-5">
							<p class="table-body">eg, Odds of DNA change and disease occurring together are <span class="table-body_symbol">&gt;</span>1&#160;in 32&#160;in a &#173;family</p>
						</td>
					</tr>
					<tr class="Basic-Table _idGenTableRowColumn-8">
						<td class="Basic-Table CellOverride-5">
							<p class="table-body"><span class="table-body_bold">Computational and predictive data</span></p>
						</td>
						<td class="Basic-Table CellOverride-5" />
						<td class="Basic-Table CellOverride-5">
							<p class="table-body">eg, Evidence that a DNA change is &#173;silent or that &#173;there is no change to gene product</p>
						</td>
						<td class="Basic-Table CellOverride-5" />
						<td class="Basic-Table CellOverride-5">
							<p class="table-body">eg, DNA change affects gene product, such as missense mutation or protein truncation</p>
						</td>
						<td class="Basic-Table CellOverride-5" />
						<td class="Basic-Table CellOverride-5">
							<p class="table-body">eg, DNA change produces an amino acid change that is established as pathogenic or as a null mutation</p>
						</td>
					</tr>
					<tr class="Basic-Table _idGenTableRowColumn-9">
						<td class="Basic-Table CellOverride-6">
							<p class="table-body"><span class="table-body_bold">Functional data; cell or animal models</span></p>
						</td>
						<td class="Basic-Table CellOverride-6" />
						<td class="Basic-Table CellOverride-6">
							<p class="table-body">eg, Well-&#173;established functional data show no deleterious effect</p>
						</td>
						<td class="Basic-Table CellOverride-6" />
						<td class="Basic-Table CellOverride-6">
							<p class="table-body">eg, Mutation hot spot in well-&#173;studied functional domain</p>
						</td>
						<td class="Basic-Table CellOverride-6" />
						<td class="Basic-Table CellOverride-6">
							<p class="table-body">eg, Well-&#173;established functional studies show a deleterious effect</p>
						</td>
					</tr>
					<tr class="Basic-Table _idGenTableRowColumn-10">
						<td class="Basic-Table CellOverride-7" colspan="7">
							<p class="table-footnote"><span class="table-footnote_italic">Note:</span> Bioinformaticians in ge&#173;ne&#173;tics laboratories use multiple types of data (population, segregation, computational, and functional) to help determine the likelihood of DNA change being a pathogenic mutation. Often, data are incomplete data in some domains. The more information &#173;there is supporting pathogenicity, the more confident the clinician can be in applying the results of ge&#173;ne&#173;tic testing to patient management in areas such as predictive DNA testing or gene therapy. Ge&#173;ne&#173;tic testing for inherited ret&#173;i&#173;nal disease is now more than 75% sensitive in detecting the causative mutation.</p>
						</td>
					</tr>
				</tbody>
			</table>
		</div>
		<div class="_idGenObjectLayout-1">
			<div id="_idContainer033">
				<div id="_idContainer031" class="Basic-Text-Frame">
					<p class="figure-caption"><span class="figure-number">Figure&#160;5-10</span> Blockade of translation by antisense oligonucleotides. Normal gene transcription of DNA into mRNA is followed by translation of mRNA into protein. Antisense oligonucleo&#173;tides&#160;complementary to a portion of mRNA bind mRNA, preventing translation—&#173;either by the steric effect of the binding pro&#173;cess itself or (possibly) by inducing degradation of the mRNA by RNase. <span class="figure-source-note">(Reproduced with permission from Askari FK, McDonnell WM. Antisense-&#173;oligonucleotide therapy.</span> <span class="figure-source-emphasis">N Engl J Med.</span> <span class="figure-source-note">1996;334(5):316–318.)</span></p>
				</div>
				<div id="_idContainer032" class="_idGenObjectStyleOverride-1">
					<img class="_idGenObjectAttribute-1" src="BCSC2019_S02_C05_p169_198_9P-web-resources/image/fig_5-10.png" alt="" />
				</div>
			</div>
		</div>
	</body>
</html>
